---
author: Qi Sun date: '2022' title: Dedifferentiation maintains melanocyte stem cells in a dynamic niche - s41586-023-05960-6.pdf
---

# Dedifferentiation maintains melanocyte stem cells in a dynamic niche - s41586-023-05960-6.pdf

![Figure from page 1](images/page_001_content/img-000.png) Article Dedifferentiation maintains melanocyte stem cells in a dynamic niche https://doi.org/10.1038/s41586-023-05960-6 Qi Sun1, Wendy Lee1, Hai Hu1, Tatsuya Ogawa1, Sophie De Leon1, Ioanna Katehis1,
                                                               Chae Ho Lim1, Makoto Takeo1, Michael Cammer2, M. Mark Taketo3, Denise L. Gay1,5,
Received: 1 July 2022
                                                               Sarah E. Millar4 & Mayumi Ito1 ✉
Accepted: 16 March 2023 Published online: 19 April 2023
                                                               For unknow reasons, the melanocyte stem cell (McSC) system fails earlier than other
Open access adult stem cell populations1, which leads to hair greying in most humans and mice2,3.
    Check for updates Current dogma states that McSCs are reserved in an undierentiated state in the hair
                                                               follicle niche, physically segregated from dierentiated progeny that migrate away
                                                               following cues of regenerative stimuli4–8. Here we show that most McSCs toggle
                                                               between transit-amplifying and stem cell states for both self-renewal and generation
                                                               of mature progeny, a mechanism fundamentally distinct from those of other self-
                                                               renewing systems. Live imaging and single-cell RNA sequencing revealed that McSCs
                                                               are mobile, translocating between hair follicle stem cell and transit-amplifying
                                                               compartments where they reversibly enter distinct dierentiation states governed
                                                               by local microenvironmental cues (for example, WNT). Long-term lineage tracing
                                                               demonstrated that the McSC system is maintained by reverted McSCs rather than by
                                                               reserved stem cells inherently exempt from reversible changes. During ageing, there
                                                               is accumulation of stranded McSCs that do not contribute to the regeneration of
                                                               melanocyte progeny. These results identify a new model whereby dedierentiation is
                                                               integral to homeostatic stem cell maintenance and suggest that modulating McSC
                                                               mobility may represent a new approach for the prevention of hair greying.
Mammalian tissue regeneration largely depends on the capacity of the bulge are postulated to represent long-term stem cells6. How- adult stem cells to differentiate. Stem cell differentiation is generally ever, their distinct functions and self-renewal capacities have yet viewed as unidirectional and follows the hierarchical model originally to be characterized. Despite the close relationship between HFSCs established through the study of haematopoietic stem cells9–11. This and McSCs, there are disparities in their durability over time: McSCs theory proposes that stem cells (undifferentiated state) have two dis- become exhausted earlier than HFSCs in most animals and humans, tinct fates: one to sustain themselves through self-renewal and the which results in hair greying during ageing1–3. The high prevalence second to produce transit-amplifying (TA) progeny (intermediate of hair greying suggests that there may be specific disadvantages in differentiated state) that ultimately give rise to functional differenti- the long-term maintenance of McSCs. ated cells during tissue regeneration12–14. In this model, the life-long durability of self-renewing tissues is typically sustained by a func- tionally and molecularly heterogeneous pool of stem and progenitor HG McSCs can regenerate all melanocyte compartments cells. To better understand the hierarchical structure of the McSC system, The organization of the McSC system, responsible for hair pig- we quantified the distribution of McSCs before HF regeneration (that mentation, is thought to parallel that of hair follicle stem cells is, in the telogen phase). McSCs are defined as DCT+ melanocytes (HFSCs)5–8. McSCs are located in the bulge and hair germ (HG) area located at the level of the bulge and slightly below the bulge area4. in telogen-phase hair follicles (HFs)4,5, where they are surrounded DCT is a melanocyte lineage marker expressed by all melanocytes by HF epithelial stem cells (bulge cells)14 and progenitor cells (HG during the hair cycle4,8,17. Consistent with previous microscopy analy- cells)15,16 that constitute to the McSC niche. At the onset of the anagen ses4,5, DCT+ McSCs were found in the bulge and the HG. However, a growth phase, McSCs regenerate differentiated melanocytes that new volumetric analysis with 3D imaging of whole telogen HFs from migrate downwards into the hair bulb, where they produce pigment Dct LacZ and Dct rtTA;tetOH2B-GFP reporter mice17,18 revealed that most HFs for the hair. Similar to HG epithelial cells, HG McSCs activate WNT lacked any McSCs within the CD34+ bulge. Rather, most McSCs were signalling and undergo differentiation at the onset of regeneration7. concentrated in the P-cadherin+ HG (Fig. 1a and Extended Data Fig. 1). Furthermore, McSCs in the bulge cycle more slowly than those in the Even in the HFs that contained rare bulge McSCs, the majority were HG during HF regeneration6. On the basis of these studies, McSCs in located in the HG (Extended Data Fig. 1a). In agreement, labelling of 1 The Ronald O. Perelman Department of Dermatology and Department of Cell Biology, New York University Grossman School of Medicine, New York, NY, USA. 2Division of Advanced Research Technologies, New York University Grossman School of Medicine, New York, NY, USA. 3Colon Cancer Program, Kyoto University Hospital–iACT, Kyoto University, Kyoto, Japan. 4Black Family Stem Cell Institute, Department of Cell, Developmental and Regenerative Biology and Department of Dermatology, Icahn School of Medicine at Mount Sinai, New York, NY, USA. 5Present address: DLGBioLogics, Paris, France. ✉e-mail: mayumi.ito@nyulangone.org 774 | Nature | Vol 616 | 27 April 2023 ``` ![Figure from page 2](images/page_002_content/img-000.png) ![Figure from page 2](images/page_002_content/img-001.png) ![Figure from page 2](images/page_002_content/img-002.png) ![Figure from page 2](images/page_002_content/img-003.png) ![Figure from page 2](images/page_002_content/img-004.png) ![Figure from page 2](images/page_002_content/img-005.png) ![Figure from page 2](images/page_002_content/img-006.png) ![Figure from page 2](images/page_002_content/img-007.png) ![Figure from page 2](images/page_002_content/img-008.png) ![Figure from page 2](images/page_002_content/img-009.png) ![Figure from page 2](images/page_002_content/img-010.png) ![Figure from page 2](images/page_002_content/img-011.png) ![Figure from page 2](images/page_002_content/img-012.png) ![Figure from page 2](images/page_002_content/img-013.png) ![Figure from page 2](images/page_002_content/img-014.png) ![Figure from page 2](images/page_002_content/img-015.png) ![Figure from page 2](images/page_002_content/img-016.png) ![Figure from page 2](images/page_002_content/img-017.png) ![Figure from page 2](images/page_002_content/img-018.png) ![Figure from page 2](images/page_002_content/img-019.png) ![Figure from page 2](images/page_002_content/img-020.png) ![Figure from page 2](images/page_002_content/img-021.png) ![Figure from page 2](images/page_002_content/img-022.png) ![Figure from page 2](images/page_002_content/img-023.png) ![Figure from page 2](images/page_002_content/img-024.png) ![Figure from page 2](images/page_002_content/img-025.png) ![Figure from page 2](images/page_002_content/img-026.png) ![Figure from page 2](images/page_002_content/img-027.png) ![Figure from page 2](images/page_002_content/img-028.png) ![Figure from page 2](images/page_002_content/img-029.png) ![Figure from page 2](images/page_002_content/img-030.png) a c 1st Telogen Early anagen Mid/late anagen Early/mid catagen Late catagen 2nd Telogen
     LacZ
                                                            6% Bg Bg
                                                                                    HFs in which all Bg
                                                      13% McSCs reside in the HG
                                                                                    HFs with 1 McSC HG
                                           Bg in the bulge HG
                                                              81% HFs with >1 McSCs
                                                                                    in the bulge tdTomato
                                         HG GFP ORSup
                                                                                                                                  Both stem and differentiated
b In vivo tracing of HG McSC TyrcreER;RosaLSL-tdTomato; progeny
                                                                                 K14rtTA;tetOH2B-GFP
                                       1st Early Mid/late Early/mid Late 2nd
                                     Telogen anagen anagen catagen catagen Telogen
                                                                                                                                                                                      Bb
                                     Initial visit
                                                     Frequent revisits of the same HF every 1–4 days
                                                                                                                            1st Telogen Early anagen Mid/late anagen Early/mid catagen Late catagen 2nd Telogen
d e Bg
                                                                                                                                                                                      Bg
                                                                                                                                                                                                                                                         Bg
                                                                                                                        HG
                                                 28%
                                                                                                                                                                                                                                                         HG
                                   58% tdTomato
                                                 14% 100% GFP
                                                                                                                                                                                      ORSup
                Stem progeny only tdTomato+ bulb melanocytes
                                                                             that die at the end of anagen
                Differentiated progeny only
                                                                             tdTomato+ bulb melanocytes
                Both stem and that survive during catagen
                differentiated progeny Stem progeny only
                                                               P = 1.7 × 10–6
f
                                   200 Bb
      Furrthest distance between
          two tdTomato+ cells
                                   150 1st Telogen Early anagen Mid/late anagen Early/mid catagen Late catagen 2nd Telogen
                                                                                                                       Bg Bg
                                   100
                                                                                                                                                                                                                                                                      Bg
                                                                                                                      HG
                                    50
                                                                                                                                                                                                                                                                     HG
                                                                                                                                                                                                             ORSup **
                                                                                                                             tdTomato
                                      0
                                                                                                                             GFP
                                                Late Early/mid Late 2nd
                                               anagen catagen catagen Telogen
                                                                                                                                                                                                               *
                                                                                                                                                                                                               *
                                                                                                                                   Differentiated progeny only
g
                                    17%
                                                                    HG
                                                                    Bulge
                                          83%
                                                                                                                                                                                                Bb
Fig. 1 | HG McSCs possess self-renewal ability. a, Left, bright-field image of a 7 mice. e, Percentages of tdTomato+ bulb MCs that die at the end of anagen X-gal-stained HF from a Dct LacZ mouse at telogen. Right, quantification of McSCs or survive during the catagen phase. N = 20 HFs from 2 mice. f, Bar graph in specified locations in telogen HFs from Dct LacZ mice. N = 52 single, whole showing the furthest distance between two tdTomato+ cells in the bulge/ HFs from 3 mice. b, Timeline of in vivo imaging of single HG McSCs in Tyr creER ; ORSup (at late anagen and early/mid catagen) and bulge/HG (at late catagen Rosa LSL-tdTomato;K14 rtTA ;tetO H2B-GFP mice. HFs containing single HG tdTomato- and second telogen). Data are presented as the mean ± s.d. P values (one-way labelled cells were identified at the first telogen stage and then revisited at analysis of variance (ANOVA) with Bonferroni multiple comparison test) are indicated stages. c, Live z-stack images of three representative examples of indicated, with 95% confidence interval at 35.81–81.13. N = 21 (late anagen), tdTomato+ HG McSCs that undergo distinct fates. K14+ epithelial cells (GFP+) 27 (early/mid catagen) or 23 (late catagen and telogen) HFs from 2 mice. are in green. See Extended Data Fig. 2 for additional examples. Yellow dashed g, Percentages of tdTomato+ McSCs residing in bulge or HG at second telogen. line outlines club hair. White asterisk marks tdTomato+ cell in unrelated HF N = 60 HFs from 6 mice. Dashed black or white lines outline the epithelial– (bottom). d, Quantification of distinct fates of HG McSCs. N = 59 HFs from dermal boundary (a,c). Scale bars, 20 μm (a) or 10 μm (c). Bb, bulb; Bg, bulge. Wnt1cre;RosaLSL-tdTomato mice, in which neural-crest-derived cells, includ- a mouse model in which McSCs are genetically labelled with tdTo- ing the melanocyte lineage, are labelled by the fluorescent protein mato, and K14+ epithelial cells express GFP, which outlines the HF tdTomato showed that most tdTomato+ McSCs were concentrated structure (Tyr creER;RosaLSL-tdTomato;K14rtTA;tetOH2B-GFP mice19–22) (Fig. 1b). in the HG7 (Extended Data Fig. 1c). These results suggest that hair Tyr creER mice are broadly utilized to target melanocytes, including pigmentation and McSC maintenance may solely rely on McSCs in DCT+ McSCs7,8,19,23,24. For clonal analysis, we treated the mice with a low the HG. dose of tamoxifen during telogen and identified HFs with only one To verify that HG McSCs have self-renewal ability, we performed labelled McSC in the HG compartment. Revisitation of the same HF fate mapping of telogen HG McSCs using in vivo imaging. We used during anagen (Methods and Extended Data Fig. 2a) showed that HG
                                                                                                                                                                                                                   Nature | Vol 616 | 27 April 2023 | 775
Article McSCs labelled during telogen gave rise to mature melanocytes in the regenerative stimuli, and McSC maintenance may rely on their dedif- hair bulb that die at the end of anagen25 (Fig. 1c–e and Extended Data ferentiation. Fig. 2b). Notably, we also discovered that labelled McSCs migrated To test this hypothesis, we sought to trace the fate of the most dif- to the bulge and upper outer root sheath (bulge/ORSup) stem cell ferentiated subset of HG McSCs. We focused on a late-pigmentation niche during anagen and exhibited self-renewal capacity that ena- gene, Oca2 (refs. 33–35), that is not expressed by quiescent telogen bled their persistence (Fig. 1c,d and Extended Data Fig. 2b). During McSCs but is upregulated in all mature bulb melanocytes based on late catagen, most McSCs in the bulge/ORSup were aggregated in the our scRNA-seq data (Fig. 2e,f). Notably, a small subset of early-anagen lower follicle (Fig. 1f, Extended Data Fig. 3). By the next telogen phase, McSCs also expressed Oca2 (Fig. 2e,f). To confirm that Oca2+ McSCs they had primarily homed back to the HG compartment (Fig. 1c,g and are more differentiated than Oca2– McSCs in early-anagen HFs, we Extended Data Fig. 2b ). These results demonstrate that unlike the reanalysed scRNA-seq data in which Oca2+ cells had diverged from HFSC system, in which HG epithelial cells disappear after proliferation other early-anagen McSCs by regressing out cell cycle genes (Fig. 2g). and differentiation15,26, melanocytes residing within the HG possess Analyses revealed that differentiation markers (that is, pigmentation self-renewal ability. genes) were expressed at higher levels in Oca2+ than in Oca2– McSCs
                                                                              (Fig. 2h). Unlike Oca2+ bulb melanocytes, the Oca2+ early-anagen McSCs
                                                                              retained expression of genes shared by telogen and other early-anagen
Trajectory of McSC differentiation at anagen onset McSCs (Extended Data Fig. 6). These results suggest that whereas most Previous studies have shown that WNT proteins in the HG promote HG melanocytes at the early-anagen phase initiate the upregulation of differentiation of epithelial cells and melanocytes following induction differentiation and pigmentation genes, the most differentiated ones of HF regeneration7. Epithelial HG cells then function as TA cells that also express Oca2 (Fig. 2i). proliferate and differentiate without self-renewal capacity15,26. To inves- We then generated Oca2creER;RosaLSL-tdTomato reporter mice as a means tigate how McSCs can retain their stemness while being aggregated in to genetically trace the fate of Oca2+ HG McSCs (Extended Data Fig. 7a). such a pro-differentiation environment, we examined the differentia- No tdTomato+ cells were observed in the HFs of mice that did not receive tion status of HG McSCs. a tamoxifen injection (Extended Data Fig. 7b). Using this new tool, In the quiescent telogen stage, McSCs displayed the compact oval we traced the fate of McSCs from anagen onset onwards. We treated or bipolar shape of undifferentiated melanocytes. However, they Oca2creER;RosaLSL-tdTomato mice with tamoxifen once daily for 3 days dur- showed marked changes during early anagen (anagen II), when TA ing early anagen (Extended Data Fig. 7c). This treatment successfully cells emerge in this HG compartment4–6, with all melanocytes in the labelled Oca2+ (tdTomato-labelled) McSCs, which displayed dendritic HG developing a dendritic appearance reminiscent of differentiated morphology, expressed Oca2 mRNA and were found in the HG (Fig. 3a melanocytes27 (Fig. 2a,b). In more than 90% of the HFs examined, all and Extended Data Fig. 7d,e). Oca2+ cells included both proliferative melanocytes had dendritic morphology during this phase (Fig. 2b). and nonproliferative McSCs (Extended Data Fig. 7f), results that are Live imaging showed that the majority of HG McSCs had transformed consistent with our scRNA-seq data (Extended Data Fig. 5d,e). into a dendritic morphology within the niche before their initial divi- Once the HF fully formed the anagen bulb, cells initially labelled sion (Fig. 2c and Extended Data Fig. 4). Daughter cells were also den- for Oca2 (tdTomato+) in the HG had contributed to both differenti- dritic, which suggested that morphological changes represent an ated melanocytes in the hair bulb and to McSCs in the anagen bulge/ early feature of McSC activation (Fig. 2c and Extended Data Fig. 4). ORSup (Fig. 3a and Extended Data Fig. 7g,h). By mid-to-late (mid/late) This morphological change was transient, and McSCs retracted anagen, tdTomato+ cells within the bulge/ORSup niche reverted to dendrites after they migrated from the HG to the bulge/ORSup an Oca2– state (Fig. 3a), and other differentiation markers (that is, (Fig. 2c,d). pigmentation genes) such as Gpr143 were downregulated (Extended Single-cell RNA sequencing (scRNA-seq) comparisons of telogen Data Fig. 7i). and early-anagen McSCs and mature bulb melanocytes purified by By the subsequent telogen phase, most tdTomato+ cells had relocated fluorescence-activated cell sorting (FACS) verified that genes involved to the HG (Extended Data Fig. 7j) and retained their undifferentiated in melanocyte dendrite formation28,29, such as Rac1, were upregulated state (Fig. 3a). These tdTomato+ McSCs retained the ability to regen- in early anagen, similar to that observed in mature bulb melanocytes erate bulb melanocytes for at least 2 years (the final time point exam- of anagen HFs (Extended Data Fig. 5a–c). Given that the definition of ined; Fig. 3b,c). In agreement with our scRNA-seq data (Extended Data differentiation in biology is “a process by which a less specialized cell Figs. 5f and 6), those HG Oca2+ McSCs destined for dedifferentiation matures to fulfil the function of the lineage”30, we examined pigmen- retained their expression of stem-cell-enriched genes (for example, tation genes as reliable differentiation markers. FeaturePlot analysis Col12a1, Txnip and Cdh1 (which encodes E-cadherin)), whereas those showed that numerous pigmentation genes, absent from telogen destined for final maturation in the bulb lost expression of these genes McSCs, were detected in most early-anagen McSCs (Fig. 2e). Further- (Extended Data Fig. 7i). more, a comparison of proliferative and nonproliferative early-anagen Lineage tracing of late-anagen Oca2+ differentiated bulb melano- McSCs revealed similar signatures, which suggested that differen- cytes confirmed the live-imaging results that bulb melanocytes do tiation may occur independent of the proliferative state (Extended not persist (Extended Data Fig. 8). These results demonstrate that Data Fig. 5d,e). Nonetheless, a distinct set of genes was expressed highly differentiated Oca2+ McSCs in the HG but not mature bulb mel- by both telogen and early-anagen HG McSCs, but downregulated in anocytes can dedifferentiate to give rise to undifferentiated McSCs bulb melanocytes (Extended Data Fig. 5f,g). Some of these genes have in the niche. potential stem cell relevance, with functions related to renewal and survival31,32. Pseudotime analysis confirmed that early-anagen McSCs reside between telogen McSCs and bulb melanocytes (Extended Data McSCs in the TA compartment can dedifferentiate Fig. 5h). Previous studies have shown that TA cells in the McSC system depend
                                                                              on c-Kit and can be eliminated after injection of a c-Kit-neutralizing
                                                                              antibody4,6,36. To determine whether Oca2+ cells depend on c-Kit,
The most differentiated McSCs can dedifferentiate we injected a c-Kit-neutralizing antibody into mice. This resulted in On the basis of these analyses, we formulated the following hypothesis. almost complete elimination of tdTomato+ cells (Fig. 3d), which sug- McSCs may differentiate into an intermediate differentiation (TA-like) gested that TA cells that emerge in early anagen include Oca2+ McSCs. state at the whole population level in the growing HG following cues of Current theories indicate that such TA cells are spatially segregated 776 | Nature | Vol 616 | 27 April 2023 ![Figure from page 4](images/page_004_content/img-000.png) ![Figure from page 4](images/page_004_content/img-001.png) ![Figure from page 4](images/page_004_content/img-002.png) ![Figure from page 4](images/page_004_content/img-003.png) ![Figure from page 4](images/page_004_content/img-004.png) ![Figure from page 4](images/page_004_content/img-005.png) ![Figure from page 4](images/page_004_content/img-006.png) ![Figure from page 4](images/page_004_content/img-007.png) ![Figure from page 4](images/page_004_content/img-008.png) ![Figure from page 4](images/page_004_content/img-009.png) ![Figure from page 4](images/page_004_content/img-010.png) ![Figure from page 4](images/page_004_content/img-011.png) ![Figure from page 4](images/page_004_content/img-012.png) ![Figure from page 4](images/page_004_content/img-013.png) ![Figure from page 4](images/page_004_content/img-014.png) ![Figure from page 4](images/page_004_content/img-015.png) ![Figure from page 4](images/page_004_content/img-016.png) ![Figure from page 4](images/page_004_content/img-017.png) ![Figure from page 4](images/page_004_content/img-018.png) ![Figure from page 4](images/page_004_content/img-019.png) ![Figure from page 4](images/page_004_content/img-020.png) ![Figure from page 4](images/page_004_content/img-021.png) ![Figure from page 4](images/page_004_content/img-022.png) ![Figure from page 4](images/page_004_content/img-023.png) ![Figure from page 4](images/page_004_content/img-024.png) ![Figure from page 4](images/page_004_content/img-025.png) ![Figure from page 4](images/page_004_content/img-026.png) ![Figure from page 4](images/page_004_content/img-027.png) ![Figure from page 4](images/page_004_content/img-028.png) ![Figure from page 4](images/page_004_content/img-029.png) ![Figure from page 4](images/page_004_content/img-030.png) ![Figure from page 4](images/page_004_content/img-031.png) ![Figure from page 4](images/page_004_content/img-032.png) ![Figure from page 4](images/page_004_content/img-033.png) ![Figure from page 4](images/page_004_content/img-034.png) ![Figure from page 4](images/page_004_content/img-035.png) ![Figure from page 4](images/page_004_content/img-036.png) ![Figure from page 4](images/page_004_content/img-037.png) ![Figure from page 4](images/page_004_content/img-038.png) ![Figure from page 4](images/page_004_content/img-039.png) ![Figure from page 4](images/page_004_content/img-040.png) ![Figure from page 4](images/page_004_content/img-041.png) a c Telogen Early anagen Mid/late anagen
                                                           Telogen Early anagen
                                                                                                                                                                                                                                                                                                                                                                                                                Bg
                     tdTomato tdTomato Bg
                                                                                                                                                                                                                                                                 HG
                                                                                                                                                                                                                                                                                                                                                                                                                       ORSup
                                                                                                                                                                                                                                                                                 tdTomato
                                                                                                                                                                                                                                                                                 GFP
                    HG
                                                                                                          HG
b 6 P = 0.0006
                                                                                                                                                                                       120 P = 0.0023
                                                                                                                                                             McSCs are dendritic (%)
                                                                                                                                                                                                                                                                                                                                                             –14
                                                                                                                                                                                                                                                                                                                                    P = 3 × 10–18 P = 5 × 10
                                No. of dendrites per
                                 tdTomato+ McSC
                                                                                                                                                                                       100 12
                                                                                                                                                                                                                                                                d
                                                                                                                                                                HFs in which all
                                                       4 80
                                                                                                                                                                                                                                                                    No. of dendrites per
                                                                                                                                                                                                                                                                                           10
                                                                                                                                                                                                                                                                      tdTomato+ cell
                                                                                                                                                                                        60 8
                                                       2 40 Bb
                                                                                                                                                                                                                                                                                            6
                                                                                                                                                                                        20
                                                                                                                                                                                                                                                                                            4
                                                       0 0
                                                                    Telogen Early Telogen Early 2
                                                                                       anagen anagen 0
                                                                                                                                                                                                                                                                                                                                 Telogen Early Mid- Late
                                                                                                                                                                                                                                                                                                                                              anagen anagen anagen
e Telogen McSC
                          Early-anagen McSC
                          Differentiated bulb melanocyte
                    20 Mlana Tyrp1 Pmel Slc24a5 Mlph Slc45a2
                    10 20 6 20 20 20 20 20
    UMAP 2
                                                                                                                                                                                                      5 5 5 3
                                                                                                                                          5
                                                                             UMAP 2
                                                                                                                                                    UMAP 2
                                                                                                                                                                                                                              UMAP 2
                                                                                                                                                                                                                                                                                           UMAP 2
                                                                                                                                                                                                                                                                                                                                                                UMAP 2
                                                                                                                                                                                                                                                                                                                                                                                                       UMAP 2
                      0 10 10 4 10 10 4 10 10 3
                                                                                                                                          4 4 2
                                                                                        0 3 0 3 0 3 0 3 0 0 2
                    −10 −10 −10 −10 −10 2 −10 −10
                                                                                                                                          2 2 2 1
                                                                                                                                                                                                      1 1 1
                    −20 −20 1 −20 −20 1 −20 −20 −20
                          −20 −10 0 10 20 −20 −10 0 10 20 0 −20 −10 0 10 20 0 −20 −10 0 10 20 0 −20 −10 0 10 20 0 −20 −10 0 10 20 0 −20 −10 0 10 20 0
                                UMAP 1 UMAP 1 UMAP 1 UMAP 1 UMAP 1 UMAP 1 UMAP 1
                            Rab38 Rab27a Mc1r Trpm1 Slc24a4 Gpr143 Oca2
                     20 20 3 20 20 20 20 20 3
                                                                                                                                                                                                                                                                                   4 2.0 3
        UMAP 2
                                                                             UMAP 2
                                                                                                                                                    UMAP 2
                                                                                                                                                                                                                             UMAP 2
                                                                                                                                                                                                                                                                                           UMAP 2
                                                                                                                                                                                                                                                                                                                                                                UMAP 2
                                                                                                                                                                                                                                                                                                                                                                                                       UMAP 2
                     10 3 10 10 2 10 10 10 10
                                                                                                                                          2 3 1.5 2 2
                      0 2 0 0 0 0 0 0
                                                                                                                                                                                                                                                                                   2 1.0
                    −10 −10 1 −10 1 −10 −10 −10 1 −10 1
                                                                         1 1 0.5
                    −20 −20 −20 −20 −20 −20 −20
                      −20 −10 0 10 20 0 −20 −10 0 10 20 0 −20 −10 0 10 20 0 −20 −10 0 10 20 0 −20 −10 0 10 20 0 −20 −10 0 10 20 0 −20 −10 0 10 20 0
                            UMAP 1 UMAP 1 UMAP 1 UMAP 1 UMAP 1 UMAP 1 UMAP 1
f g Oca2– cluster h i Single-cell McSC composition
                                                                              Oca2+ cluster differentiation trajectory in the niche
                                                                                                                                                                                                                                                                                                    Slc24a4 expression level
                                                                                                                                                                                                          Gpr143 expression level
                                                                                                          Oca2 expression level
                    100 Naive Undifferentiated
                                                                                                                                           2.0 2.0 1.5 McSC Telogen HG
                                                                     5
                                                           UMAP 2
                     80 1.5 1.5 1.0
                                                                                                                                           1.0 1.0
Oca2+ cells (%)
                                                                     0 0.5
                                                                                                                                           0.5 0.5
                     60 0 0 0
                                                                    −5 Oca2– early-
                                                                                                                                                                      Oca2– Oca2+ Oca2– Oca2+ Oca2– Oca2+ anagen McSC
                     40 −5 0 5 Early-anagen stem cell Early-anagen stem cell Early-anagen stem cell
                                                                                 UMAP 1
                     20
                                                                                                                                                                                                                   Slc24a5 expression level
                                                                                                                       Trpm1 expression level
                                                                                                                                                                                                                                                                                                        Pmel expression level
                      0 Oca2 Oca2+ early-
                                                                    10 anagen McSC Early-anagen HG
                                                                                                    2.0 4 4 5
                                                           UMAP 2
                                                                     5 3 3 4
                                                                                                    1.5 3
                    Telogen McSC 2 2 2
                                                                     0 1.0 1 1 1
                    Early-anagen −5 0.5 0 0 0
                    McSC Oca2– Oca2+ Oca2– Oca2+ Oca2– Oca2+
                                                                                                    0
                    Differentiated −5 0 5 Differentiated
                                                                                                                                                             Early-anagen stem cell Early-anagen stem cell Early-anagen stem cell
                    bulb melanocyte UMAP 1 Differentiated melanocyte
Fig. 2 | McSCs in the HG initiate a differentiation programme during early presented as the mean ± s.d. P values derived using one-way ANOVA with anagen. a, z-stack images of tdTomato+ McSCs from Tyr creER ;Rosa LSL-tdTomato mice Bonferroni multiple comparison test, with the following 95% confidence at telogen and early anagen. Denoted areas are magnified and reconstructed intervals: −4.804 to −3.122 (telogen versus early anagen); 2.430 to 4.112 using Imaris (magenta, tdTomato+; green, DAPI). b, Left, number of dendrites (early anagen versus mid-anagen). e, Uniform manifold approximation and per tdTomato+ McSCs (N = 3 mice, ≥30 tdTomato+ McSCs analysed per mouse). projection (UMAP) plot of merged FACS-isolated telogen McSCs, early-anagen Right, percentages of HFs containing only dendritic McSCs (N = 3 mice, 20 HFs McSCs and differentiated anagen VI bulb melanocytes. FeaturePlots show analysed per mouse). Melanocytes with ≥3 dendrites were considered dendritic. expression of pigmentation genes. f, Percentages of Oca2+ cells within each Data are presented as the mean ± s.d. P values derived by two-tailed unpaired population. g, UMAP plot of early-anagen McSCs after regression of cell cycle t-test. c, Live revisits of a representative first telogen HF containing a single HG genes. The FeaturePlot shows Oca2 expression. h, Violin plots showing tdTomato-labelled cell in a Tyr CreER ;Rosa LSL-tdTomato;K14 rtTA ;tetO H2B-GFP mouse. differential expression of pigmentation genes in early-anagen Oca2+ and Oca2− K14+ epithelial cells (GFP+) are in green. Bottom panels show magnified views. clusters. i, Schematic of melanocyte differentiation trajectory and McSC d, Dendrite number per tdTomato+ McSCs in HG (telogen and early anagen) and composition in the niche. Dashed white lines outline the epithelial–dermal bulge/ORSup (mid-anagen and late anagen). N = 27 HFs from 2 mice. Data are boundary (a,c). Scale bars, 20 μm (a) or 10 μm (c). in the lower part of the HG during early anagen5–8. We used in vivo live was genetically labelled in the lowermost portion of the growing imaging to verify whether Oca2+ McSCs in this area could still revert HG and bulb (in anagen II/IIIa HFs of Oca2creER;RosaLSL-tdTomato;K14rtTA; to a stem cell state. We identified HFs in which a single Oca2+ McSC tetOH2B-GFP mice). Revisiting these HFs at mid/late anagen revealed that
                                                                                                                                                                                                                                                                                                                                                                         Nature | Vol 616 | 27 April 2023 | 777
![Figure from page 5](images/page_005_content/img-000.png) ![Figure from page 5](images/page_005_content/img-001.png) ![Figure from page 5](images/page_005_content/img-002.png) ![Figure from page 5](images/page_005_content/img-003.png) ![Figure from page 5](images/page_005_content/img-004.png) ![Figure from page 5](images/page_005_content/img-005.png) ![Figure from page 5](images/page_005_content/img-006.png) ![Figure from page 5](images/page_005_content/img-007.png) ![Figure from page 5](images/page_005_content/img-008.png) ![Figure from page 5](images/page_005_content/img-009.png) ![Figure from page 5](images/page_005_content/img-010.png) ![Figure from page 5](images/page_005_content/img-011.png) ![Figure from page 5](images/page_005_content/img-012.png) ![Figure from page 5](images/page_005_content/img-013.png) ![Figure from page 5](images/page_005_content/img-014.png) ![Figure from page 5](images/page_005_content/img-015.png) ![Figure from page 5](images/page_005_content/img-016.png) ![Figure from page 5](images/page_005_content/img-017.png) ![Figure from page 5](images/page_005_content/img-018.png) ![Figure from page 5](images/page_005_content/img-019.png) ![Figure from page 5](images/page_005_content/img-020.png) ![Figure from page 5](images/page_005_content/img-021.png) ![Figure from page 5](images/page_005_content/img-022.png) ![Figure from page 5](images/page_005_content/img-023.png) ![Figure from page 5](images/page_005_content/img-024.png) ![Figure from page 5](images/page_005_content/img-025.png) ![Figure from page 5](images/page_005_content/img-026.png) ![Figure from page 5](images/page_005_content/img-027.png) ![Figure from page 5](images/page_005_content/img-028.png) ![Figure from page 5](images/page_005_content/img-029.png) Article a Anagen onset Mid/late anagen Telogen
                                                                                                                                   ORSup Bulb
               tdTomato tdTomato IF tdTomato tdTomato IF tdTomato IF tdTomato tdTomato IF
                                                  Oca2 FISH Oca2 FISH Oca2 FISH Oca2 FISH
                                                                                                Bg ORSup
                                                                            tdTomato IF tdTomato IF
                                                  HG Oca2 FISH Oca2 FISH
                                                                                                                                                                              HG HG
              HG
                                                                                                          Bb
                                                                                                                                                                                                               c-Kit
b c d antibody
                                                                                                                                                                                         Control
                                                                           2 years telogen 2 years anagen VI tdTomato tdTomato
                     5 Bg Bg
                                           P = 0.3309 P = 0.0156
                                                                                                                                                                                                                               No. of bulge/ORSup tdTomato+
                                                                           tdTomato tdTomato 5
                     4 Oca2creER;RosaLSL-tdTomato mice
No. of tdTomato+
                                                                                                                                                                                                    ORSup
                                                                                                                                                                                                                                                              4
cells per area
                                                                                                                                                                                                                                       cells per area
                     3 Day 0 1 3 5 7 (mid-anagen)
                                                                                                                                                                                                                      ORS up
                                                                                                                                                                                                                                                              3
                                                                                                                                                 Tamoxifen
                     2 Bb
                                                                                                                                                                                                                                                              2
                                                                                                                                 Depilation Analysis
                     1 c-Kit c-Kit c-Kit 1
                                                                           HG antibody antibody antibody
                     0 or PBS or PBS or PBS 0
                          Anagen Telogen 2 years of Bb Control c-Kit
                           onset telogen Bb
                                                                                                                                                                                                                                                                              antibody
e f Fate of lowermost
                             Both stem and differentiated cell Stem cell only Differentiated cell only Oca2+ McSC
        Anagen onset Mid/late anagen Telogen Anagen onset Mid/late anagen Telogen Anagen onset Mid/late anagen Telogen
        tdTomato
                                                                                           Bg
        GFP Bg Bg Bg Bg 31%
                                                                                      Bg Bg
                                                                                                                 Bg Bg Bg
        HG 54%
                                                               ORSup
                                                                                                                                                                       ORS
                                                                                           HG
                                                                                                                      OR u
                                                                                      HG 15%
                                                                                                                                         HG HG
                                                                                                                                                                         up
                                                                                                                        S p
                                          HG HG
                                                                                                                                                                       * * Fate of upper HG
                                                                                                                                                                                                                                                    Oca2+ McSC
                                    tdTomato+
                                    melanocyte 23%
                                    Other Bb 54%
                                    melanocyte 23%
                                                                                                                                                                                     Bb
                                                                                                                                                                                                                                       Both
                                                  Bb Stem cell progeny only
                                                                                                                                                                                                                                       Differentiated progeny only
Fig. 3 | Oca2+ cells located in the TA compartment can undergo mice treated with PBS (control) or a c-Kit-neutralizing antibody. Right, images dedifferentiation during anagen. a–c, Oca2 creER ;Rosa LSL-tdTomato mice were and quantification of tdTomato + McSCs. N = 3 mice. Ten areas were analysed injected with tamoxifen three times during depilation-induced anagen per mouse. P value derived by two-tailed unpaired t-test. e, Live lineage onset for Oca2+ cell lineage tracing. a, Detection of tdTomato and Oca2 tracing of a single tdTomato + cell of Oca2 creER :Rosa LSL-tdTomato;K14 rtTA :tetO H2B-GFP (fluorescence in situ hybridization (FISH)) as indicated. Denoted areas of mice reveals three fates. GFP marks K14+ epithelial cells. Cartoon illustrates tdTomato-only detection are reconstructed using Imaris (magenta, tdTomato +; the relative locations of tdTomato + melanocytes and other melanocytes in green, DAPI). Magnified views of tdTomato immunofluorescence (IF) and the HF. White asterisks indicate tdTomato+ cells in an unrelated HF. f, Percentages OCA2 FISH are represented in single colour. b, Number of tdTomato + cells of single Oca2+ cells in the lowest HG region (N = 13 HFs from 1 mouse) or over time. N = 5 mice (anagen onset and telogen) and N = 3 mice (2-year telogen). the upper HG (N = 13 HFs from 1 mouse) that give rise to specific progeny as Five areas were analysed per mouse. P values were derived using one-way ANOVA defined. Insets in mid/late anagen images in a and c show bright-field images with Bonferroni multiple comparison test, with 95% confidence intervals of of the bulb. For b and d, data are presented as the mean ± s.d. Scale bars, −0.7397 to 1.993. c, Detection of tdTomato in telogen and induced anagen at 20 μm or 10 μm (single colour images of a). Dashed white lines outline the 2 years following tamoxifen treatment. d, Left, timeline of injections and epithelial–dermal boundary (a,c,d,e). analysis 7 days after depilation of tdTomato expression in Oca2 creER ;Rosa LSL-tdTomato Oca2+ McSCs in the lower HG were able to translocate to the bulge/ HF or whether other McSCs can also express Oca2 during a subsequent ORSup, the stem cell compartment of the HF and relocated to the HG by anagen phase. To test this theory, we repeatedly induced HF regenera- the next telogen, similar to upper HG Oca2+ McSCs (Fig. 3e,f). Alter- tion and labelled Oca2+ McSCs at each anagen onset through tamoxifen natively, they could also descend with HF downgrowth and mature treatment (Extended Data Fig. 9a). We observed gradual increases in into fully differentiated melanocytes in the bulb (Fig. 3e,f). These the percentage of HFs containing tdTomato+ cells and in the number results demonstrate that late-stage Oca2+ McSCs positioned in the of tdTomato+ McSCs per HF (Extended Data Fig. 9b–e). This result con- lowermost part of the growing HG, known as the TA compartment, trasted with control experiments, in which Oca2+ McSCs were labelled can still dedifferentiate and persist long term. only once. These results show that new (unlabelled) McSCs can express
                                                                                                                              Oca2 at different anagen onsets (approximately 20% at one hair cycle
                                                                                                                              based on scRNA-seq analysis) and contribute to both fates. After three
No McSC is inherently exempted from differentiation cycles of repeated tamoxifen injection, HF niches in which residents As not all McSCs express Oca2 during the early-anagen phase, we asked were exclusively tdTomato+ McSCs (>20% HFs analysed) were observed whether the Oca2+ subset might constitute a fixed subpopulation in the (Extended Data Fig. 9f), which demonstrated that different McSCs 778 | Nature | Vol 616 | 27 April 2023 express Oca2 in different hair cycles. These results suggest that there cells survived and inappropriately continued along the differentia- is no reserved population of stem cells that are inherently exempt from tion pathway to express pigment after translocation into the anagen experiencing Oca2+ TA-like status (Extended Data Fig. 9g). bulge/ORSup (Extended Data Fig. 12a,b). By contrast, normal Oca2+
                                                                            cells reverted to an undifferentiated state (Extended Data Fig. 12a,b).
                                                                            Thus, downregulation of WNT activity permits the dedifferentiation
Pigmented McSCs can dedifferentiate of Oca2+ McSCs. When followed up long term, tdTomato+ McSCs in the Given that the skin is constantly exposed to environmental factors that β-catenin stabilized mice showed a significant reduction (Extended promote melanocyte differentiation37,38, we asked whether McSCs can Data Fig. 12c), a result consistent with previous work demonstrating still dedifferentiate in such an environment. To that end, we irradiated that long-term WNT activation depletes McSCs7. Oca2creER;RosaLSL-tdTomato mice with UVB light and injected tamoxifen to Last, to confirm that WNT active status is regulated by epithelial label Oca2+ cells with the tdTomato reporter (Extended Data Fig. 10a). niche cells, we used K15crePR1;Wls fl/fl mice to inducibly delete Wntless Two days after treatment, tdTomato+ McSCs produced abundant pig- (Wls) and prevent WNT ligand release specifically from K15+ epithelial ment in the stem cell niche (Extended Data Fig. 10b,c). UVB-treated cells that surround McSCs in the bulge and HG7,44–46. Following induc- McSCs expanded (Extended Data Fig. 10b,c) and expressed differ- tion with RU486 and subsequent anagen onset, these mice exhibited entiation markers in the mid-anagen bulge/ORSup, albeit to a lesser defects in nuclear β-catenin expression in McSCs and in surrounding extent compared with bulb melanocytes (Extended Data Fig. 10d). epithelial cells (Fig. 4a). This result suggested that epithelial-derived These results suggest that UVB irradiation induces McSCs to acceler- WNT ligands have an essential role in McSC WNT activation. To ensure ate their differentiation into functional melanocytes. By late anagen, that only McSCs were disrupted in WNT activity, we also examined tdTomato+ McSCs located in the bulge/ORSup had significantly down- McSCs in RU486-treated K15crePR1;Wls fl/fl;Ctnnb1STA mice43, in which Wls is regulated expression of the differentiation markers Tyrp1, Oca2 and deleted but β-catenin signalling is retained in epithelial cells. Previously Gpr143, which suggested that they were undergoing dedifferentiation published results7 and Fig. 4b show that constitutive WNT activation (Extended Data Fig. 10d,e). By the next telogen phase, tdTomato+ McSCs in control RU486-treated K15crePR1;Ctnnb1STA HFSCs resulted in aber- still homed back to the HG. They regained an undifferentiated pheno- rant but pigmented HF development. However, in mice lacking Wls, type, as evidenced by the downmodulation of differentiation markers epithelial niche cells showed WNT activation and nuclear β-catenin and pigmentation (Extended Data Fig. 10f). Two years after UVB irra- localization, despite their inability to secrete WNT ligands (Fig. 4b). diation, comparable numbers of tdTomato+ McSCs were observed in By contrast, DCT+ melanocytes did not exhibit nuclear β-catenin, they the HG and pigment was absent (Extended Data Fig. 10g). tdTomato+ did not express the melanocyte differentiation marker MITF or pro- McSCs remained competent to generate bulb MCs capable of hair pig- duce pigment (Fig. 4b). These results demonstrate the requirement mentation following depilation-induced HF regeneration (Extended of epithelial WNT ligands for McSC WNT activation. Together, these Data Fig. 10h). As observed for untreated Oca2+ McSCs (Extended Data findings show that niche-derived WNT ligands regulate the reversible Fig. 7i), the UVB-treated Oca2+ McSCs destined for dedifferentiation differentiation of McSCs. retained expression of stem-cell-enriched genes (for example, Col12a1, Txnip and E-cadherin) (Extended Data Fig. 10i). These results show that McSCs that are driven to fulfil their ultimate differentiated function of Repetitive plucking increases dormant bulge McSCs producing visible pigment following UVB irradiation can still revert to The above results suggest that the ability of McSCs to translocate an undifferentiated state (Extended Data Fig. 10j). between the stem cell (WNT– bulge/ORSup) and TA (WNT+ HG/bulb)
                                                                            compartments of the HF may enable them to reversibly attain distinct
                                                                            differentiation states. However, our live imaging showed that not all
Dynamic niche regulates McSC reversibility McSCs originating from the HG home back to the HG compartment at We next investigated the mechanisms that enable McSC dedifferentia- the next telogen phase but rather remain in the telogen bulge (Figs. 1c,g tion. Previous studies have shown that constitutive activation of WNT and 3e and Extended Data Fig. 7j). Given the high prevalence of hair in McSCs leads to their untimely differentiation and eventual loss, greying in humans and mice, we investigated whether this shift in dis- whereas inhibition of WNT leads to deficient melanocyte regenera- tribution of HG McSCs might correlate with ageing. tion7,39,40. WNT ligands are upregulated in the HG at anagen onset to To this end, we experimentally accelerated the ageing of HFs by promote McSC differentiation7,15,41,42. As the HG cells grow down to repeated depilation of Dct LacZ mice and Dct rtTA;tetOH2B-GFP mice at every form the HF bulb, the WNT active area also moves down as the anagen telogen phase starting from postnatal day 21 (P21)17,18,47. Consistent phase progresses (Extended Data Fig. 11a). Nonetheless, our live imag- with previous studies1, we detected HFs with significant loss of McSCs, ing demonstrated that HG McSCs can translocate to the anagen bulge/ which was evident in the HG by the seventh telogen phase (Fig. 4c). As ORSup (Figs. 1c and 3e and Extended Data Fig. 2b), where WNT signal- expected, these mice displayed hair greying (Fig. 4c). Notably, many ling is known to be suppressed. We proposed that downregulation of McSCs in aged HFs had changed location and were scattered to the WNT ligands following migration to the bulge/ORSup may permit the bulge rather than being tightly aggregated within the HG compartment dedifferentiation of McSCs. (Fig. 4d and Extended Data Fig. 13a,b). Image analyses revealed that First, to understand how WNT status correlates with differentiation the distance between individual McSCs was greater in aged HFs than status, we examined scRNA-seq data. The analysis showed that WNT in young HFs (Fig. 4d). The percentage of HFs containing bulge McSCs activation genes were upregulated during early anagen, especially in increased from 10% to more than 50% (Fig. 4d). A similar observation late-stage Oca2+ McSCs (Extended Data Fig. 11b,c). Immunofluores- was noted when we examined Oca2creER;RosaLSL-tdTomato mice following cence data confirmed that Oca2+ McSCs activate WNT signalling in repeated plucking (≥6 hair cycles) (Extended Data Fig. 13c–e). anagen onset HG (Extended Data Fig. 11d–f). Notably, WNT signalling To understand whether such abnormal McSC distribution can affect was subsequently downregulated by mid/late anagen in the bulge/ the regeneration of mature melanocytes, we specifically traced the rare ORSup regardless of UVB irradiation (Extended Data Fig. 11d–f). bulge McSCs in young telogen HFs through live imaging and assessed Second, we verified whether persistent WNT activation in ana- their ability to generate differentiated melanocytes. Telogen bulge gen Oca2+ McSCs precludes reversion to an undifferentiated McSC McSCs gave rise only to bulge/ORSup McSC progeny, without contribut- state. Constitutive activation of WNT signalling by expressing stabi- ing to the differentiated melanocyte compartment (Fig. 4e,f). Moreo- lized β-catenin (encoded by Ctnnb1) in Oca2+ McSCs (from Oca2creER; ver, bulge McSCs either remained quiescent or underwent limited Ctnnb1STA;RosaLSL-tdTomato mice43) showed that in this environment, these division to give rise to McSC progeny (Fig. 4e,f). This was in contrast
                                                                                                              Nature | Vol 616 | 27 April 2023 | 779
![Figure from page 7](images/page_007_content/img-000.png) ![Figure from page 7](images/page_007_content/img-001.png) ![Figure from page 7](images/page_007_content/img-002.png) ![Figure from page 7](images/page_007_content/img-003.png) ![Figure from page 7](images/page_007_content/img-004.png) ![Figure from page 7](images/page_007_content/img-005.png) ![Figure from page 7](images/page_007_content/img-006.png) ![Figure from page 7](images/page_007_content/img-007.png) ![Figure from page 7](images/page_007_content/img-008.png) ![Figure from page 7](images/page_007_content/img-009.png) ![Figure from page 7](images/page_007_content/img-010.png) ![Figure from page 7](images/page_007_content/img-011.png) ![Figure from page 7](images/page_007_content/img-012.png) ![Figure from page 7](images/page_007_content/img-013.png) ![Figure from page 7](images/page_007_content/img-014.png) ![Figure from page 7](images/page_007_content/img-015.png) ![Figure from page 7](images/page_007_content/img-016.png) Article a b P22 RU486 10 days
                                                                       Day –7 0 3
                                                                                                                                                                                                                                                                                                                                   RU486
                                               K15crePR1;Wls/ mice RU486 K15crePR1;Ctnnb1STA mice and
                                                and control mice K15crePR1;Wls/;Ctnnb1STA mice Analysis
                                                                                                                                                                                    Depilation Analysis 80
                                                                                                                                                                                                                                                                                                                                           Nuclear β-catenin+DCT+ cells (%)
                                       Control K15crePR1;Wls/ K15crePR1;Ctnnb1STA P = 0.0333
                                                                                                               Nuclear β-catenin+DCT+ cells (%)
                   β-catenin 100
                                                                                                                                                                                                                          β-catenin
                   DCT P = 0.0270 60
                                                                                                                                                                                                                          DCT
                                                                                                                                                           80
                                                                                                                                                           60 40
                                                                                                                                                           40 20
        HG HG 20
                                                                                                                                                                                                                                                                                                                                                                                                         0
                                                                                                                                                                 0 K15crePR1; K15crePR1;
                                                                                                                                                                             Control K15crePR1; Ctnnb1STA Wls/;
                                                                                                                                                                                      Wls/ Ctnnb1STA
                                                                                                                                                                                                                                                                 K15crePR1;Wls/;Ctnnb1STA
                                                                                                                                                                                                                                                                                                                                                                                                                        P = 0.0011
c 2nd Telogen 7th Telogen
                                                                                                                                                                                                                          β-catenin MITF
                                                                                                                                                                                                                                                                                                                                                                                                         80
                                                                                                                                                                                                                          DCT DCT
                                                                                                                                                                                                                                                                                                                                                                              Pigmented DCT+ cells (%)
                                                                                                                                                                        15 P = 0.0292
                                                                                                                                                                                                                                                                                                                                                                                                         60
                                                                                                                                                  McSCs in the HG (%)
                                                                                                                                                   HFs that depleted
                                                                                                                                                                        10 40
                                                                                                                                                                                                                                                                                                                                                                                                         20
                                                                                                                                                                         5
                                                                                                                                                                                                                                                                                                                                                                                                             0
                                                                                                                                                                                                                                                                                                                                                                                                                  K15crePR1; K15crePR1;
                                                                                                                                                                         0 Ctnnb1STA Wls/;
                                                                                                                                                                                 2nd 7th Ctnnb1STA
                                                                                                                                                                               Telogen Telogen
                                                                                                                                                                                                           e Telogen Early anagen Mid/late anagen
d 2nd Telogen 7th Telogen Bg Bg
                                      GFP GFP
                                                                                                                                                                                                                                             HG
                                                                                                                                                                                                                                                                                                                                                                                                                                     ORSup
                                      Bg tdTomato
                                                                                                                                                                                                                                                   GFP
                                                                                                                                                                                    Bg
                                       HG
                                                                                                               HG
                                                                                                                                                                                                           f
                                                                                                                                                                                                                                                                                                        12 P = 3.8 × 10–8
                                                                                                                                                                        P = 0.0006
                                                       P = 0.0426 P = 1.8 × 10–5
                                                                                                               60
                                                                              HFs containing bulge McSCs (%)
      Average distance of each McSC
                                       25 10
                                                                                                                                                                                                                                                                              McSC by mid/late anagen
                                                                                                                                                                                                                                      100
          to the centre of HG (μm)
                                                                                                                                                                                                          McSCs giving rise to bulb
                                                                                                                                                                                                                                                                                No. of divisions per
                                       20 8
                                                                                                                                                                                                                                       80
                                                                                                                                                                                                             melanocytes (%)
                                                                                                               40
                                       15 6 Bb
                                                                                                                                                                                                                                       60
                                       10 40 4
                                                                                                               20
                                           5 20 2
                                           0 0 0 0
                                                     2nd 7th 2nd 7th Bulge McSC HG McSC Bulge McSC HG McSC
                                                   Telogen Telogen Telogen Telogen
Fig. 4 | HF ageing limits the access of McSCs to niche signals that reversibly McSCs in GFP. Right, average distance of each McSC from the HG centre regulate their differentiation. a, Top, K15 crePR1 ;Wls fl/fl conditional knockout (second telogen, N = 3 mice; seventh telogen, N = 4 mice. Five HFs analysed per mice and control mice (7 weeks old) were treated with RU486 and depilated. mouse). Right, percentages of HFs containing bulge McSCs (second telogen, Bottom left, immunofluorescence of DCT and β-catenin at 3 days after N = 3 mice, 20 HFs analysed per mouse; seventh telogen, N = 4 mice, ≥36 HFs depilation. Bottom right, percentages of DCT+ McSCs with nuclear β-catenin. analysed per mouse). e, Live tracing of a single tdTomato+ bulge McSC in N = 3 mice. b, Top, K15 crePR1 ;Ctnnb1 STA mice and K15 crePR1 ;Wls fl/fl;Ctnnb1 STA mice a Tyr creER ;Rosa LSL-tdTomato;K14 rtTA ;tetO H2B-GFP mouse. f, Left, percentages of bulge were treated with RU486 from P22. Bottom left, immunofluorescence of DCT, and HG McSCs that produced differentiated progeny (HG, N = 7 mice, 59 HFs; β–catenin and MITF with corresponding bright-field images. Bottom right, bulge, N = 6 mice, 8 HFs). Right, the number of divisions by each bulge and HG percentages of DCT+ McSCs with nuclear β-catenin signals and pigmentation. McSC by mid/late anagen (N = 35 HFs with a single HG McSC; 8 HFs with a single N = 3 mice. c,d, Comparison of Dct rtTA ;tetO H2B-GFP mice at second and seventh bulge McSC). For a–d,f, data are presented as the mean ± s.d. P values derived telogen. c, Representative images of mice. Percentages of HFs lacking HG McSC by two-tailed unpaired t-test. Dashed white or red lines outline the epithelial– (second telogen: N = 3 mice, 20 HFs analysed per mouse. Seventh telogen: dermal boundary (a,b,d,e). Scale bars, 20 μm (a,b,d), 10 μm (e) or 1 cm (c). N = 4 mice, ≥36 HFs analysed per mouse). d, Left, z-stack images of HFs with to HG McSCs, which actively proliferated to regenerate progeny in both stem cell and differentiated cell compartments (Figs. 1c,d and 4f). Discussion These results reveal that telogen bulge McSCs, which do not directly Collectively, our data demonstrate that the McSC system is a thin- contribute to bulb melanocyte regeneration, increase during HF ageing layered system in which HG McSCs act as both stem cells and TA cells induced by repetitive hair regeneration. at the onset of regeneration. The findings reveal a new model whereby 780 | Nature | Vol 616 | 27 April 2023 dedifferentiation plays an integral part in homeostatic stem cell main- 8. Infarinato, N. R. et al. BMP signaling: at the gate between activated melanocyte stem cells
                                                                                                             and differentiation. Genes Dev. 34, 1713–1734 (2020).
tenance (Extended Data Fig. 11f). 9. Seita, J. & Weissman, I. L. Hematopoietic stem cell: self-renewal versus differentiation. This new model of McSC maintenance highlights a previously Wiley Interdiscip. Rev. Syst. Biol. Med. 2, 640–653 (2010). unknown level of plasticity. We also identified the vulnerability of 10. Orkin, S. H. & Zon, L. I. Hematopoiesis: an evolving paradigm for stem cell biology. Cell
                                                                                                             132, 631–644 (2008).
this system compared with most other stem cell systems that operate 11. Morrison, S. J., Uchida, N. & Weissman, I. L. The biology of hematopoietic stem cells. through multilayered distinct stem cell and progenitor cell popula- Annu. Rev. Cell Dev. Biol. 11, 35–71 (1995). tions with separate functions and locations. McSCs need to be mobile 12. Cockburn, K. et al. Gradual differentiation uncoupled from cell cycle exit generates
                                                                                                             heterogeneity in the epidermal stem cell layer. Nat. Cell Biol. 24, 1692–1700 (2022).
to demonstrate their chameleon-like features, whereby they display 13. Barker, N. Adult intestinal stem cells: critical drivers of epithelial homeostasis and a TA phenotype when located in the TA compartment of the HF (the regeneration. Nat. Rev. Mol. Cell Biol. 15, 19–33 (2014). growing HG area) and then a stem cell phenotype once translocated 14. Cotsarelis, G., Sun, T. T. & Lavker, R. M. Label-retaining cells reside in the bulge area
                                                                                                             of pilosebaceous unit: implications for follicular stem cells, hair cycle, and skin
into the stem cell compartment of the HF (Extended Data Fig. 11f). carcinogenesis. Cell 61, 1329–1337 (1990). Therefore, HFSC and McSC organization during tissue regeneration 15. Greco, V. et al. A two-step mechanism for stem cell activation during hair regeneration. may appear to be parallel when stem cell and TA cells are defined by Cell Stem Cell 4, 155–169 (2009).
                                                                                                       16. Silver, A. F. & Chase, H. B. DNA synthesis in the adult hair germ during dormancy (telogen)
their differentiation phenotype. The crucial difference between these and activation (early anagen). Dev. Biol. 21, 440–451 (1970). two stem cell systems is that McSCs, which exhibit the TA phenotype 17. Zaidi, M. R. et al. Interferon-γ links ultraviolet radiation to melanomagenesis in mice. in regard to differentiation status and location, can move back and Nature 469, 548–553 (2011).
                                                                                                       18. Zhao, S. & Overbeek, P. A. Tyrosinase-related protein 2 promoter targets transgene
dedifferentiate into stem cells. However, we also demonstrated that expression to ocular and neural crest-derived tissues. Dev. Biol. 216, 154–163 (1999). McSC movement between the stem cell and TA compartments is not 19. Bosenberg, M. et al. Characterization of melanocyte-speciic inducible Cre recombinase precisely recapitulated during repeated regenerations, resulting transgenic mice. Genesis 44, 262–267 (2006).
                                                                                                       20. Madisen, L. et al. A robust and high-throughput Cre reporting and characterization system
in the accumulation of McSCs that fail to undergo this movement for the whole mouse brain. Nat. Neurosci. 13, 133–140 (2010). during HF ageing. 21. Nguyen, H., Rendl, M. & Fuchs, E. Tcf3 governs stem cell features and represses cell fate Previous studies have demonstrated that positions of individual determination in skin. Cell 127, 171–183 (2006).
                                                                                                       22. Tumbar, T. et al. Deining the epithelial stem cell niche in skin. Science 303, 359–363
adult stem cells within the niche are unfixed or interchangeable in the (2004). HFSC and intestinal systems12,26,48. Because McSCs regenerate hair mel- 23. Moon, H. et al. Melanocyte stem cell activation and translocation Initiate cutaneous
                                                                                                             melanoma in response to UV exposure. Cell Stem Cell 21, 665–678.e6 (2017).
anocytes only when they are within the HG compartment, their proper
                                                                                                       24. Sun, Q. et al. A novel mouse model demonstrates that oncogenic melanocyte stem
localization would be required to prevent hair greying. Current theory cells engender melanoma resembling human disease. Nat. Commun. 10, 5023 proposes that McSCs accumulate genotoxic damage during ageing and (2019).
                                                                                                       25. Tobin, D. J., Hagen, E., Botchkarev, V. A. & Paus, R. Do hair bulb melanocytes undergo
are gradually eliminated over time through terminal differentiation in
                                                                                                             apoptosis during hair follicle regression (catagen)? J. Invest. Dermatol. 111, 941–947
response to anagen niche signals49. As such, pro-differentiation signals (1998). are likely to be concentrated in the HG compartment7, and bulge McSCs, 26. Rompolas, P., Mesa, K. R. & Greco, V. Spatial organization within a niche as a determinant
                                                                                                             of stem-cell fate. Nature 502, 513–518 (2013).
when they exist, may have a higher survival rate owing to reduced expo-
                                                                                                       27. Hirobe, T. How are proliferation and differentiation of melanocytes regulated? Pigment
sure to such differentiation stimuli. Under extraordinary conditions, Cell Melanoma Res. 24, 462–478 (2011). bulge McSCs might act as a valid source of melanocyte production 28. Kim, M. Y. et al. MKK6 increases the melanocyte dendricity through the regulation of Rho
                                                                                                             family GTPases. J. Dermatol. Sci. 60, 114–119 (2010).
for hair re-pigmentation, but relocation closer to the HG would be a
                                                                                                       29. Scott, G. Rac and Rho: the story behind melanocyte dendrite formation. Pigment Cell
prerequisite for this. Res. 15, 322–330 (2002). Last, tumours derived from melanocytes (melanomas) retain self- 30. Paigen, K. The genetic control of enzyme activity during differentiation. Proc. Natl. Acad.
                                                                                                             Sci. USA 47, 1641–1649 (1961).
renewal ability regardless of their fully differentiated, pigmented phe- 31. Jung, H. et al. TXNIP maintains the hematopoietic cell pool by switching the function of notype50,51, which is unlike many other tumours, including epithelial p53 under oxidative stress. Cell Metab. 18, 75–85 (2013). cancers. Because of this difference, melanomas are difficult to eliminate 32. Fan, G. C. Role of heat shock proteins in stem cell behavior. Prog. Mol. Biol. Transl. Sci. 111,
                                                                                                             305–322 (2012).
and represent the most dangerous form of skin cancer. This study has 33. Le, L., Sirés-Campos, J., Raposo, G., Delevoye, C. & Marks, M. S. Melanosome biogenesis provided evidence for the plasticity of normal McSCs under physi- in the pigmentation of mammalian skin. Integr. Comp. Biol. 61, 1517–1545 (2021). ological conditions. An implication of this finding is that this marked 34. Manga, P., Boissy, R. E., Piko-Hirst, S., Zhou, B. K. & Orlow, S. J. Mislocalization of
                                                                                                             melanosomal proteins in melanocytes from mice with oculocutaneous albinism type 2.
capacity of normal melanocytes to retain self-renewal ability after Exp. Eye Res. 72, 695–710 (2001). undergoing differentiation could at least partially underlie the plastic 35. Rinchik, E. M. et al. A gene for the mouse pink-eyed dilution locus and for human type II nature of melanoma. oculocutaneous albinism. Nature 361, 72–76 (1993).
                                                                                                       36. Botchkareva, N. V., Khlgatian, M., Longley, B. J., Botchkarev, V. A. & Gilchrest, B. A. SCF/c-kit
                                                                                                             signaling is required for cyclic regeneration of the hair pigmentation unit. FASEB J. 15,
                                                                                                             645–658 (2001).
Online content 37. Yamada, T. et al. Wnt/β-catenin and kit signaling sequentially regulate melanocyte stem
                                                                                                             cell differentiation in UVB-induced epidermal pigmentation. J. Invest. Dermatol. 133,
Any methods, additional references, Nature Portfolio reporting summa- 2753–2762 (2013). ries, source data, extended data, supplementary information, acknowl- 38. Chou, W. C. et al. Direct migration of follicular melanocyte stem cells to the epidermis edgements, peer review information; details of author contributions after wounding or UVB irradiation is dependent on Mc1r signaling. Nat. Med. 19, 924–929
                                                                                                             (2013).
and competing interests; and statements of data and code availability 39. Sun, Q. et al. Dissecting Wnt signaling for melanocyte regulation during wound healing. are available at https://doi.org/10.1038/s41586-023-05960-6. J. Invest. Dermatol. 138, 1591–1600 (2018).
                                                                                                       40. Takeo, M. et al. EdnrB governs regenerative response of melanocyte stem cells by crosstalk
                                                                                                             with Wnt signaling. Cell Rep. 15, 1291–1302 (2016).
1. Nishimura, E. K., Granter, S. R. & Fisher, D. E. Mechanisms of hair graying: incomplete 41. Reddy, S. et al. Characterization of Wnt gene expression in developing and postnatal
     melanocyte stem cell maintenance in the niche. Science 307, 720–724 (2005). hair follicles and identiication of Wnt5a as a target of Sonic hedgehog in hair follicle
2. Keogh, E. V. & Walsh, R. J. Rate of greying of human hair. Nature 207, 877–878 (1965). morphogenesis. Mech. Dev. 107, 69–82 (2001).
3. O’Sullivan, J. D. B. et al. The biology of human hair greying. Biol. Rev. Camb. Philos. Soc. 42. Choi, Y. S. et al. Distinct functions for Wnt/β-catenin in hair follicle stem cell proliferation
     96, 107–128 (2021). and survival and interfollicular epidermal homeostasis. Cell Stem Cell 13, 720–733
4. Nishimura, E. K. et al. Dominant role of the niche in melanocyte stem-cell fate (2013).
     determination. Nature 416, 854–860 (2002). 43. Harada, N. et al. Intestinal polyposis in mice with a dominant stable mutation of the
5. Nishimura, E. K. Melanocyte stem cells: a melanocyte reservoir in hair follicles for hair β-catenin gene. EMBO J. 18, 5931–5942 (1999).
     and skin pigmentation. Pigment cell & melanoma research 24, 401–410 (2011). 44. Morris, R. J. et al. Capturing and proiling adult hair follicle stem cells. Nat. Biotechnol. 22,
6. Ueno, M., Aoto, T., Mohri, Y., Yokozeki, H. & Nishimura, E. K. Coupling of the radiosensitivity 411–417 (2004).
     of melanocyte stem cells to their dormancy during the hair cycle. Pigment Cell Melanoma 45. Myung, P. S., Takeo, M., Ito, M. & Atit, R. P. Epithelial Wnt ligand secretion is required for
     Res. 27, 540–551 (2014). adult hair follicle growth and regeneration. J. Invest. Dermatol. 133, 31–41 (2013).
7. Rabbani, P. et al. Coordinated activation of Wnt in epithelial and melanocyte stem cells 46. Carpenter, A. C., Rao, S., Wells, J. M., Campbell, K. & Lang, R. A. Generation of mice with a
     initiates pigmented hair regeneration. Cell 145, 941–955 (2011). conditional null allele for Wntless. Genesis 48, 554–558 (2010).
                                                                                                                                                     Nature | Vol 616 | 27 April 2023 | 781
Article
47. Endou, M., Aoki, H., Kobayashi, T. & Kunisada, T. Prevention of hair graying by factors that Open Access This article is licensed under a Creative Commons Attribution
    promote the growth and differentiation of melanocytes. J. Dermatol. 41, 716–723 (2014). 4.0 International License, which permits use, sharing, adaptation, distribution
48. Zhang, C. et al. Escape of hair follicle stem cells causes stem cell exhaustion during aging. and reproduction in any medium or format, as long as you give appropriate
    Nat. Aging 1, 889–903 (2021). credit to the original author(s) and the source, provide a link to the Creative Commons licence,
49. Inomata, K. et al. Genotoxic stress abrogates renewal of melanocyte stem cells by and indicate if changes were made. The images or other third party material in this article are
    triggering their differentiation. Cell 137, 1088–1099 (2009). included in the article’s Creative Commons licence, unless indicated otherwise in a credit line
50. Damsky, W. E. et al. β-catenin signaling controls metastasis in Braf-activated Pten-deicient to the material. If material is not included in the article’s Creative Commons licence and your
    melanomas. Cancer Cell 20, 741–754 (2011). intended use is not permitted by statutory regulation or exceeds the permitted use, you will
51. Rambow, F., Marine, J. C. & Goding, C. R. Melanoma plasticity and phenotypic diversity: need to obtain permission directly from the copyright holder. To view a copy of this licence,
    therapeutic barriers and opportunities. Genes Dev. 33, 1295–1318 (2019). visit http://creativecommons.org/licenses/by/4.0/.
Publisher’s note Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional afiliations. © The Author(s) 2023 782 | Nature | Vol 616 | 27 April 2023 Methods for 4 days before euthanizing the mice for cell isolation. In the c-Kit
                                                                                 antibody injection experiment, mice were subcutaneously injected
Generation of Oca2creER knock-in mice with 150 μl of 0.5 mg ml–1 c-Kit antibody (ACK45, BD Pharmingen) into Oca2creER knock-in line was generated following well-established pro- a 2 × 2cm area of back skin. Control mice were subcutaneously injected tocols with slight modifications52,53. The Oca2creER-targeting DNA was with 150 μl of PBS. constructed as presented in Extended Data Fig. 7a by VectorBuilder. The purified plasmid DNA of the targeting vector was linearized using Melanocyte stem cell in vivo imaging the restriction enzymes NotI and SalI for embryonic stem (ES) cell tar- In vivo imaging of melanocyte stem cells was based on previously geting. The mouse MK6 (C57BL/6J) ES cells (established at New York described methods of live imaging of HF stem cells54,55. Tyr creER; University (NYU) Langone’s Rodent Genetic Engineering Laboratory) RosaLSL-tdTomato;K14rtTA;tetOH2B-GFP mice were given a single injection were grown, at passage 6, on mitotically inactivated mouse embryonic of 60 μg tamoxifen on P21 and maintained on a 1 g kg–1 doxycycline- fibroblast (MEF) cells (Sigma Millipore) and passaged the day before containing diet from P21. All the in vivo imaging was performed at electroporation. Linearized targeting vector DNA (25 μg ml–1) contain- least 3 days after tamoxifen induction. As the timing of the hair cycle ing the neomycin-resistance gene was added to the cell suspension, is slightly variable between individual mice and individual HFs, we kept and electroporation was performed using either a Gene Pulser II or imaging the mouse to monitor the hair cycle stages. HF stages were a Gene Pulser system (Bio-Rad). Following electroporation, the cells determined according to a previous publication56 by observing the were plated onto neomycin-resistant MEFs and incubated at 37 °C, 95% pattern of GFP-labelled HF epithelial cells. We performed the initial humidity and 5% CO2. After 24 h, geneticin (160 μg ml–1 active concentra- imaging at telogen/anagen I to capture HFs that contain only a single tion, Invitrogen) was added to the growth medium for positive selec- tdTomato-labelled cell. To trace the fate of HG melanocytes, we focused tion of antibiotic-resistant ES cell colonies. The medium was changed on HFs containing a single tdTomato+ melanocyte in the HG and revis- every day, and geneticin (G418) selection was maintained for 6 days. ited the same HFs every 1–4 days to capture hair cycle stages, includ- Antibiotic-resistant ES cell colonies were counted, picked and split to ing early anagen, mid/late anagen, early-to-mid (early/mid) catagen, grow in 96-well plates duplicated for cryopreservation, and homolo- late catagen and the subsequent (second) telogen. Some mice were gous recombination events were identified through genotyping by revisited less frequently to focus on capturing mid/late anagen and Southern blot analysis. The mouse MK6 ES cells and MEFs were tested the subsequent telogen. To trace the fate of bulge McSCs, we initially for mycoplasma contamination before use and were not authenticated. visited HFs containing a single tdTomato+ McSC in the bulge at telogen ES cells with successful homologous recombination were injected and revisited the same HFs until mid/late anagen. into mouse blastocyst embryos. ES cells were trypsinized to obtain To trace the fate of melanocytes located in the TA compartment of a single-cell suspension, and the cell suspension was kept on ice in early-anagen HFs, we placed Oca2creER;RosaLSL-tdTomato;K14ktTA;tetOH2B-GFP 1 ml ES cell medium in a 15 ml tube until use. Blastocyst embryos were mice on a doxycycline-containing diet from P21 and kept imaging the collected from C57BL/6-albino females (4 weeks old, NIH 562, CRL) mouse to monitor the hair cycle stages. The mice were given a single at 3.5 days post coitum. Ten to fifteen ES cells were injected into each injection of 1.2 mg tamoxifen when the HF stage progressed to anagen blastocyst embryo, and injected blastocysts were cultured in KSOM I/early-anagen II. We then performed the initial imaging at anagen medium at 37 °C in an atmosphere of 5% CO2 for 2 h until the blastocyst II/anagen IIIa (at least 3 days after tamoxifen injection) to capture HFs cavity was recovered. The microinjected blastocysts were transferred that contained only a single tdTomato-labelled cell, which is located to the uterine horns of 2.5 days post coitum pseudopregnant females in the lowermost part of the HG. The same HFs were revisited at mid/ (CD-1, CRL) using the standard procedure to generate chimeric mice. late anagen and telogen. The chimeric mice were bred with C57BL/6 to obtain Oca2creERT2 mice. Throughout the course of imaging, mice were on a warming pad
                                                                                 and anaesthetized with vapourized isoflurane delivered through
Mouse experiments a nose cone (1.5% in oxygen and air). The ear was immobilized on a All animal experiments were performed in compliance with all relevant custom-made stage, and a glass coverslip was placed directly against it. ethical regulations for animal testing and research and in accordance An Olympus Fluoview multiphoton microscope (FVMPE-RS) equipped with animal protocols approved by the Institutional Animal Care and with a MaiTai HP DS-OL laser tuned to 940 nm and an Insight X3-OL Use Committee at NYU School of Medicine. Mice were housed in an tuned to 1,120 nm (Newport Spectraphysics) were used in line sequen- animal room with a temperature range of 20–22 °C, humidity range tial mode to acquire z-stacks with a ×25 NA. of 30–70% and under a 12–12 h dark–light cycle. Tyr creER (012328), RosaLSL-tdTomato (007905), Wnt1cre, K15crePR1 (005249), Revisitation of the same HFs in live imaging Wls fl/fl (012888) and K14rtTA (008099) mice were purchased from The To ensure successful revisitation of the same HFs, patterns of HF clus- Jackson Laboratory. Dct rtTA;tetOH2B-GFP (iDCT-GFP) mice were obtained ters and blood vessel locations were used as landmarks. Blood vessel from the NCI Mouse Repository. Dct lacZ mice were from P. Overbeek. location in the ear was first used to broadly return to the same area. Then Ctnnb1fl(ex3)/+ (Ctnnb1STA) mice were from M. M. Taketo43. Mice were HF cluster patterns in large areas were recorded using multiple tiled bred and crossed in-house to obtain experimental and control animals images (up to 30) with z-stack steps of 20–30 μm during the initial visit in mixed backgrounds. Mice from experimental and control groups and revisits. Rather than being evenly distributed, HFs in the imaged were randomly selected from either sex for experiments. Data collec- areas showed distinct clustered patterns. A cluster usually contained tion and analyses were not performed blind to the conditions of the three to ten HFs with a gap between different clusters. By numbering experiments. the clusters and finding the same clusters over time using a tiled map, To induce Cre recombination, tamoxifen (Sigma-Aldrich) treatment we frequently revisited the same cluster. Owing to the low tamoxifen was performed as previously published7 by intraperitoneal injection concentration used to achieve clonal labelling of McSCs, only a subset (0.1 mg g–1 body weight) of a 20 mg ml–1 solution in corn oil per day. For of HFs within a cluster contained any tdTomato-labelled cells. We veri- the UVB experiment, dorsal fur of 3-week-old Oca2creER;RosaLSL-tdTomato fied that within a cluster, the number and pattern of HFs that contained mice was clipped, and mice were anaesthetized. Mice were treated every tdTomato-labelled cells matched previous visits to a specific HF. other day with 600 mJ cm–2 UVB per day 3 times in total. Skin biopsies were taken from euthanized mice or under isoflurane anaesthesia. For X-gal staining isolation of McSCs and differentiated bulb MCs, DctrtTA;tetOH2B-GFP mice X-gal staining was done as previously published38. Dorsal skin from Dct lacZ were administered doxycycline-containing chow (1 g kg–1, Bio-Serv) mice was collected, and subcutaneous fat was removed using scalpel Article blades (Miltex). Tissues were fixed in 4% paraformaldehyde (PFA) for Gpr143 (ACDBio, 535108), Col12a1 (ACDBio, 312638) and Txnip (ACDBio, 30 min at 4 °C, and whole-mount X-gal staining was performed. After 457228) for the Leica System were used, and a RNAscope LS Multi- X-gal staining, skin samples were fixed again in 4% PFA at 4 °C overnight. plex Fluorescent assay (ACDBio) was used for detection. The double The tissues were then subjected to 3D whole-mount analyses. detection of immunofluorescence was performed after finishing in situ
                                                                                  hybridization by applying the primary and secondary antibodies as
3D whole-mount niche analyses described in the section ‘Immunofluorescence’. Skin sections were 3D whole-mount niche analysis was done as previously published38. then scanned using Vectra Polaris (Akoya Biosciences) at ×20 for Opal Whole skin samples of Dct LacZ mice were stained with X-gal and sequen- fluors 570 (Akoya Biosciences, FP1488001KT), 690 (Akoya Biosciences, tially treated with 25% glycerol–PBS, 50% glycerol–PBS and 100% glyc- FP1497001KT) and DAPI (Akoya Biosciences, FP1490). Scanned images erol for 3 h at room temperature or overnight at 4 °C. Skin was then cut were processed using Inform (v.2.6.0) software (Aloya Biosciences). For into thin strips using a scalpel blade. Single HFs were then dissected quantitative analysis, HALO (v.3.5) software (module Indica Labs-FISH and isolated using a dissection microscope (Axiovision Discovery V12). v.3.2.3) was utilized. FISH probe cell intensity (average intensity of FISH Skin samples of Oca2creER;RosaLSL-Tomato mice, Tyr creER;RosaLSL-Tomato mice probe (×) spots and clusters per cell) was measured in tdTomato+ cells and Dct rtTA;tetOH2B-GFP mice were collected and fixed in 4% PFA for 30 min in the mid-anagen bulge/ORSup and the bulb. The relative intensity of at room temperature to preserve tdTomato signals. Tissues were then bulge/ORSup tdTomato+ cells to bulb tdTomato+ cells within the same incubated in 30% sucrose at 4 °C overnight and embedded in OCT sample was compared across multiple samples. compound (Sakura). Then 70–100-μM-thick skin sections, including whole-mount HFs, were made and counterstained with 4′,6-diamidine- Microscopy 2′-phenylindole dihydrochloride (DAPI). For thin sections, images were taken at a single focal plane. For whole- Whole HFs were then imaged as described in the section ‘Microscopy’. mount tissues and thick sections, serial z-images were collected
                                                                                  throughout the depth of each entire HF. Wide-field fluorescence images
Immunofluorescence were taken with standard narrow-pass filters with an Eclipse Ti inverted Immunofluorescence was done as previously published but with slight microscope (Nikon) or an upright Axioplan (Zeiss). Images were pro- modifications7,24. Skin tissues were fixed overnight in 4% PFA at 4 °C. cessed to reconstruct a focused image using ImageJ/Fiji, Adobe Pho- After sequential dehydration in increasing concentrations of ethanol toshop and the extended depth of focus function in the NIS-Elements and xylene, tissues were embedded in paraffin. Sections were cut at software (Nikon, v.5.20.02). 6 μm, deparaffinized and microwaved in 10 mM Tris and 1 mM EDTA The z-stack fluorescent images in Figs. 2a and 3b were taken using (pH 8.0) for antigen retrieval. Tissue sections were then incubated a LSM 880 confocal microscope with a ×63 NA/1.4 Plan Apochromat with the following primary antibodies for 2 h at room temperature or lens (Zeiss). The z-stacks were taken at 0.3 μm steps. Volumes were overnight at 4 °C in PBT (PBS plus 0.1% Triton-X100) with 10% FBS, fol- reconstructed using Imaris 9.5 software (Oxford Instruments). lowed by Alexa-488-conjugated or Alexa-594-conjugated secondary antibodies (1:200; Thermo Fisher): goat anti-Dct (1:100; Santa Cruz, Single-cell dissociation sc-10451), rabbit anti-Tomato (1:1,000; Rockland, 600–401–379); Single melanocyte isolation was performed as previously published24 mouse anti-Tomato (RF5R) (1:500; Thermo Fisher, MA5–15257); rabbit but with slight modifications. To isolate a single melanocyte from ana- anti-Tyrp1 (1:100; Sigma-Aldrich, SAB2102617); mouse anti-E-cadherin gen II HFs, Dct rtTA;tetOH2B-GFP mice were depilated at 8 weeks old and (1:100; BD Transduction, 610181); rabbit anti-Ki67 (1:100; Abcam, fed a doxycycline-containing diet (1 g kg–1) for 4 days. At 4 days after ab15580); mouse anti-β-catenin (1:400; Sigma-Aldrich, C7207); and depilation, when the HFs are at anagen II, mice were killed. The mice mouse anti-MITF (1:100; Abcam, ab12039). The following secondary were then rinsed in betadine followed by 70% ethanol. The back skin antibodies from Thermo Fisher were used: Alexa Fluor 594 donkey of mice was collected. Scalpel blades were used to remove subcutane- anti-mouse IgG (1:200; A21203); Alexa Fluor 488 donkey anti-mouse ous fat, and skin was rinsed in PBS and cut into 1 × 1 cm pieces followed IgG (1:200; A21202); Alexa Fluor 594 donkey anti-rabbit IgG (1:200; by incubation in 0.25% trypsin for 1 h 30 min at 37 °C. Epidermis was A21207); Alexa Fluor 488 donkey anti-rabbit IgG (1:200; A21206); Alexa separated from the dermis using forceps and scalpel blades, and the Fluor 594 donkey anti-goat IgG (1:200; 11058); and Alexa Fluor 488 don- epidermis was finely chopped and incubated in 0.25% trypsin for 30 min key anti-goat IgG (1:200; A11055). Skin sections were counterstained at 37 °C while shaking at 100 r.p.m., followed by gentle pipetting to with DAPI. obtain a single-cell suspension. The obtained McSC suspension was For detection of CD34, P-cadherin and DCT, skin tissues were col- filtered through a 70 μm nylon filter and centrifuged at 200 r.c.f. for lected, processed and embedded in OCT as described above (‘3D 5 min and resuspended in medium A (DMEM, 10% FBS and 1× penicillin– Whole mount niche analyses’) to preserve tdTomato or GFP signals. streptomycin). Next, 50–70-μM-thick skin sections were made. The skin sections were To isolate single bulb melanocytes, Dct rtTA;tetOH2B-GFP were fed a incubated in PBS plus 0.5% Triton-X100 for 1 h at room temperature doxycycline-containing diet for 4 days at 5 weeks old, when the HFs and incubated with primary antibody against DCT (1:100; Santa Cruz, are at the anagen VI stage. Isolation of bulb melanocytes was performed sc-10451), CD34 (1:50; BD Pharmingen, 553731) and P-cadherin (1:100; according to previously described methods57 but with slight modi- Invitrogen, 13-2000Z) overnight at 4 °C. Skin sections were then washed fications. Mice were euthanized and rinsed in betadine followed by 3 times with PBS for 10 min at room temperature. For DCT, sections 70% ethanol. Skin was cut into 0.5 cm2 sections and incubated in 5 mM were incubated with Alexa-488-conjugated secondary antibody (1:200) EDTA (pH 8) and PBS for 2 h at 37 °C. Following incubation, connective for 1 h at room temperature. For CD34 and P-cadherin, sections were tissue, the adipocyte layer and dermis were removed using forceps, incubated with biotinylated anti-rat IgG (1:100; Vector Laboratories, and 0.5 cm2 skin samples were cut into single rows of HFs and kept in BA-9400) for 1 h followed by incubation with streptavidin Alexa 647 medium A. HF bulbs were microdissected with a surgical blade, col- conjugate (1:200, Invitrogen, S32357) for 30 min at room temperature. lected into medium A and centrifuged for 5 min at 200 r.c.f. Medium A Skin sections were counterstained with DAPI. was removed and hair bulbs were incubated in 1 ml of 0.2% collagenase II
                                                                                  and 50 U ml–1 dispase (9:1 solution) and shaken at 100 r.p.m. for 25 min
In situ hybridization at 37 °C. Next 400 U ml–1 DNase I was added and incubated for 5 min RNAscope in situ hybridization was performed using a Leica Bond at room temperature. Five volumes of medium A was added and then III automated staining platform (Leica Biosystems) according to the filtered through a 100 μm cell strainer. Cell suspension was pelleted by manufacturer’s protocol. Mouse probes of Oca2 (ACDBio, 1072511), centrifuging at 200 r.c.f. for 5 min at 4 °C and resuspended in medium A. Single GFP+ and DAPI-excluded live melanocytes were then isolated To estimate the cell cycle stage of a cell, we used Seurat’s cell cycle by cell sorting on a Sony SY3200 cell sorter equipped with a WinList scoring. In brief, averaged relative expression of cell cycle related genes 3D Analyzer (v.8.0) with a 100 μm nozzle. Single-cell suspensions from were used to calculate G2/M and S scores, which were used for binning four mice of each condition were combined for subsequent scRNA-seq cells into G2/M, S and G1/G0 bins. The cell cycle scores were regressed analyses. FlowJo 10.8.2 (Mac only) was used to plot the FACS gating out in ScaleData step in the analysis of the anagen II melanocyte dataset strategy. alone, followed by PC calculation, UMAP dimension reduction and
                                                                            unbiased clustering as described above.
scRNA-seq and data analysis Pseudotime analysis was performed using the slingshot package Single melanocyte suspensions were loaded on a 10x Genomics Chro- (v.2.2.0)60. The Seurat object was imported into slingshot using the mium instrument to generate single-cell gel beads in emulsion (GEMs). as.SingleCellExperiment function. Then a pseudotime trajectory was Approximately 5,000–10,000 cells were loaded per channel. The constructed using the slingshot function with UMAP dimensional scRNA-seq library for differentiated bulb melanocytes was prepared reduction. using the following Chromium Single Cell 3′ v2 reagent kits: Chromium Single Cell 3′ Library & Gel Bead kit v2 PN-120237; Single Cell 3′ Chip Gene set enrichment analysis kit v2 PN-120236; and i7 Multiplex kit PN-120262 (10x Genomics). The Genes differentially expressed between anagen II melanocytes com- Single Cell 3′ Reagent kits v2 User Guide (Manual Part CG00052 RevA) pared with telogen melanocytes (P < 0.05, log2(fold change) > 0.25) was followed58. The scRNA-seq library for anagen II melanocytes was were rank-ordered from high to low on the basis of their fold change. prepared using the following Chromium Single Cell 3′ v3 reagent kits: The pre-ranked gene list as queried for its enrichment in two annotated Chromium Single Cell 3′ Library & Gel Bead kit v3 PN-1000075; Single gene sets acquired from The Molecular Signature Database (MSigDB)— Cell 3ʹ Feature Barcode Library kit PN-1000079; Single Cell B Chip Kit GOBP_DENDRITE_DEVELOPMENT and GOBP_DENDRITE_MORPHOGEN- PN-1000073; and i7 Multiplex Kit PN-120262 (10x Genomics). The Single ESIS—using the preranked gene set enrichment analysis (GSEA) analysis Cell 3′ Reagent kits v3 User Guide (Manual Part CG000201 RevA) was tool61,62. A false discover rate q-value of <0.25 was deemed significant. followed. Libraries were run on an Illumina NovaSeq 6000. The Cell Ranger Single-Cell software suite (v.6.0.1) was used to perform sample Quantification and statistical analyses de-multiplexing, barcode processing and single-cell 3′ gene counting. The measurement of quantifications can be found in the y axes of bar The cDNA insert was aligned to the mm10/GRCm38 reference genome plots in the figure and in figure legends. The statistical details of each (mm10-2020-A). The undifferentiated telogen melanocyte dataset plot can be found in the figure (N number). The exact meaning of N is was downloaded from the NCBI Gene Expression Omnibus (identifier described in the corresponding figure legend. No statistical methods GSE113502) and re-processed with the same version of the Cell Ranger were used to predetermine sample sizes, but our sample sizes were Single-Cell Software Suite (v.6.0.1) and mapped to the same version of similar to those reported in previous publications7,24,38–40,55. Pairwise reference genome (mm10-2020-A). Further analysis and visualization comparisons between two groups were performed using two-tailed were performed using Seurat package (v.4.1.0)59, using R Studio Desktop unpaired t-test. Comparisons of multiple groups were performed using (v.1.4.1717) and R (v.4.1.2). one-way analysis of variance (ANOVA) or two-way ANOVA followed by The Seurat object for each condition was generated from digital gene multiple comparison test. Details of the statistical test are specified expression matrices. In the quality control step, the parameter of subset in the figure legends. Statistical significances were considered sig- cells is nFeature_RNA (200–2000) for telogen melanocytes, nCount_ nificant if P < 0.05. Exact P values are indicated in the figures and leg- RNA>10000 for anagen II melanocytes and nCount_RNA > 5000 for ends. Experimental data are shown as the mean ± standard deviation bulb melanocytes and percentage of mitochondria genes < 0.05 for all or mean ± standard error of the mean. Statistical analyses and plotting conditions. Different thresholds in filtering out low-quality cells were were done using GraphPad Prism (v.9.2.0) and Microsoft Excel (v.2016). set for each condition owing to the differences of sequencing depth among the conditions. Data were then log scaled, centred and normal- Material availability ized to the number of Unique Molecular Identifier (nUMI). Principal Materials generated in this study can be provided upon reasonable components (PCs) were calculated using Seurat’s RunPCA function. requests by contacting the corresponding author. The top 3,000 variable genes were used for calculating PCs. For each condition, UMAP dimension reduction was performed on Reporting summary the normalized, centred, scaled nUMI count matrices using the first Further information on research design is available in the Nature Port- ten PCs. We then performed unsupervised clustering using the Seurat folio Reporting Summary linked to this article. SNN clustering package, using a resolution of 0.6. Most clusters were identified as melanocytes based on expression for Dct, whereas each condition contained a minor cluster of epidermal cells positive for Krt10 Data availability or Krt14. The epidermal cells were excluded from subsequent analysis. All RNA-seq data reported in this paper are deposited into the NCBI For comparative analysis, we merged melanocyte matrices from Gene Expression Omnibus database. The accession number is three conditions (telogen melanocyte, anagen II melanocyte and dif- GSE203051. The telogen melanocyte dataset, which was previously ferentiated melanocyte) followed by a standard workflow of Seurat. deposited, has the identifier GSE113502. The mm10/GRCm38 reference Data were log normalized, scaled and centred after regressing out the genome (mm10-2020-A) utilized in the scRNA-seq analysis is available effect of nCount_RNA (the total number of molecules detected within a at https://support.10xgenomics.com/single-cell-gene-expression/ cell from sequencing) and percent.mt (mitochondria ratio, defined by software/release-notes/build#mm10_2020A. The GOBP_DENDRITE_ the PercentageFeatureSet function in Seurat). As the three conditions DEVELOPMENT dataset used in the GSEA analysis was downloaded showed differences in sequencing depth (nCount_RNA varies greatly) from the MSigDB (http://www.gsea-msigdb.org/gsea/msigdb/index. and sequencing depth is a major cause of batch effects in scRNA-seq, jsp) at http://www.gsea-msigdb.org/gsea/msigdb/mouse/geneset/ the effect of nCount_RNA was regressed out before identifying PCs GOBP_DENDRITE_DEVELOPMENT.html. The GOBP_DENDRITE_MOR- to account for such batch effects. The top 3,000 variable genes were PHOGENESIS dataset used in the GSEA analysis was downloaded from used for calculating PCs. UMAP dimension reduction was performed the MSigDB at http://www.gsea-msigdb.org/gsea/msigdb/mouse/ on the normalized, centred, scaled nUMI count matrices using the geneset/GOBP_DENDRITE_MORPHOGENESIS.html. Source data are first three PCs. provided with this paper. Article
                                                                                                       Acknowledgements We thank V. Greco and P. Myung at Yale University who enabled us to
Code availability examine melanocytes using in vivo imaging; P. Manga and S. J. Orlow for their help and support
                                                                                                       on pigmentation biology; staff at the NYU Langone’s Microscope Laboratory for assistance
All the bioinformatics analyses were performed using the following with microscopy; S. Y. Kim at the Rodent Genetic Engineering Laboratory of NYU Langone for previously published packages with publicly available codes: Cell generating the Oca2creER knock-in line; staff at the NYU Langone’s Genome Technology Center
                                                                                                       for expert assistance with library preparation and Illumina sequencing; past and present
Ranger Single-Cell software (https://support.10xgenomics.com/ members of the Ito Laboratory for their technical help and intellectual input; V. N. Nivsarkar single-cell-gene-expression/software/pipelines/latest/what-is-cell- and A. Brinks for help in formatting the manuscript; J. Walczyk for making the customized ranger); Seurat (https://satijalab.org/seurat/); slingshot (https://bio- platform for in vivo imaging; and members of NYU Langone’s Experimental Pathology Research
                                                                                                       Laboratory (RRID:SCR_017928), particularly S. Selvaraj and C. A. Loomis, for in situ hybridization.
conductor.org/packages/devel/bioc/vignettes/slingshot/inst/doc/ The cell sorting and low cytometry technologies were provided by NYU Langone’s Cytometry vignette.html); and GSEA Broad Institute (https://software.broadin- and Cell Sorting Laboratory. These centres are partially supported by a Cancer Center Support stitute.org/gsea/index.jsp). Grant (P30CA016087) at NYU Langone’s Laura and Isaac Perlmutter Cancer Center and funds
                                                                                                       from the Shared Instrumentation Grant (S10 OD021747). M.I. appreciates support from the
                                                                                                       National Institute of Arthritis and Musculoskeletal and Skin Diseases of the National Institutes
52. Bradley, A. in Teratocarcinomas and Embryonic Stem Cells, a Practical Approach of Health (R01 AR059768, R01 AR074995 and U54CA263001-01A1), the Department of Defense
    (ed. Robertson, E. J.) 113–151 (IRL Press, 1987). (W81XWH2110435 and W81XWH-21-1-0510), and the Orbuch & Brand Pilot Grant Program for
53. Nagy, A., Gertsenstein, M., Vintersten & K., Behringer, R. in Manipulating the Mouse Diseases of the Skin. Q.S. was supported by a Melanoma Research Alliance Dermatology
    Embryo, a Laboratory Manual 453–506 (Cold Spring Harbor Laboratory Press, 2003). Fellows Award (654765 and 812413) and by a NYSTEM institutional training grant (contract
54. Pineda, C. M. et al. Intravital imaging of hair follicle regeneration in the mouse. Nat. Protoc. C026880). The content is solely the responsibility of the authors and does not necessarily
    10, 1116–1130 (2015). represent the oficial views of the National Institutes of Health.
55. Rompolas, P. et al. Live imaging of stem cell and progeny behaviour in physiological
    hair-follicle regeneration. Nature 487, 496–499 (2012). Author contributions Q.S. designed and performed experiments, supervised experiments,
56. Müller-Röver, S. et al. A comprehensive guide for the accurate classiication of murine interpreted data, worked on igures and wrote the manuscript. W.L., H.H., T.O., S.D.L., I.K.,
    hair follicles in distinct hair cycle stages. J. Invest. Dermatol. 117, 3–15 (2001). C.H.L. and M.T. designed and/or performed experiments and interpreted data. M.C. helped
57. Nishikawa-Torikai, S., Osawa, M. & Nishikawa, S. Functional characterization of with the imaging analyses. D.L.G. interpreted data and edited the manuscript. M.M.T. and
    melanocyte stem cells in hair follicles. J. Invest. Dermatol. 131, 2358–2367 (2011). S.E.M. provided genetic mouse models to study WNT signalling and helped with the data
58. Zheng, G. X. et al. Massively parallel digital transcriptional proiling of single cells. interpretation. M.I. designed and supervised the project and wrote the manuscript.
    Nat. Commun. 8, 14049 (2017).
59. Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573–3587.e29 Competing interests The authors declare no competing interests.
    (2021).
60. Street, K. et al. Slingshot: cell lineage and pseudotime inference for single-cell Additional information
    transcriptomics. BMC Genomics 19, 477 (2018). Supplementary information The online version contains supplementary material available at
61. Mootha, V. K. et al. PGC-1α-responsive genes involved in oxidative phosphorylation are https://doi.org/10.1038/s41586-023-05960-6.
    coordinately downregulated in human diabetes. Nat. Genet. 34, 267–273 (2003). Correspondence and requests for materials should be addressed to Mayumi Ito.
62. Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach Peer review information Nature thanks Nick Barker, Rui Yi and the other, anonymous,
    for interpreting genome-wide expression proiles. Proc. Natl Acad. Sci. USA 102, reviewer(s) for their contribution to the peer review of this work.
    15545–15550 (2005). Reprints and permissions information is available at http://www.nature.com/reprints.
![Figure from page 14](images/page_014_content/img-000.png) Extended Data Fig. 1 | McSCs primarily reside in P-cadherin+ HG and Dct-rtTA; tetO-H2B-GFP mice. HG is considered P-cadherin+ and bulge is occasionally in CD34+ bulge at telogen. (a) Z-stack image of GFP+ cells considered P-cadherin−. N = 42 whole HFs from 3 mice. (c) Focused image of showing P-cadherin or CD34 expression on whole telogen HFs of Dct-rtTA; Tomato + cells in telogen HF of 8 weeks old Wnt1-Cre; Rosa-LSL-Tomato mouse tetO-H2B-GFP mice. Schematics depict observed distribution of GFP+ cells. (2nd telogen). HG: hair germ; Bg: bulge. Scale bar: 20 μm. Dashed white lines All hair types were included by examining hair follicles without bias per given outline epithelial-dermal boundary. Solid circle suggests location of dermal areas. (b) Quantification of the locations of McSCs in the telogen HFs from papilla. White asterisk marks Tomato signal outside of the hair follicle. ![Figure from page 15](images/page_015_content/img-000.png) Article Extended Data Fig. 2 | See next page for caption. Extended Data Fig. 2 | Additional live images of McSC lineage tracing. imaged as illustrated in Fig. 1b. A single tdTomato-labeled cell within each HF (a) Representative example of large tile images showing the pattern of HF was initially visited at the first telogen stage; then revisited at early anagen, distribution at each visit of the same area to guide repeated accurate tracing mid/late anagen, catagen and the next telogen (2nd telogen). Shown are of the same HF. Z-stack images show GFP signal (K14+ epithelium) in Tyr-CreER; representative examples of Tom+ HG McSCs that undergo self-renewal and Rosa-LSL-tdTomato;K14-rtTA;tetO-H2B-GFP mice. Orange dashed lines mark differentiation (top panels), self-renewal (middle panels) and differentiation hair follicle bundles (labeled, B1, B2, B3…). Overlapping patterns were used to (bottom panels). In the middle panels, note that one Tom+ cell died at the identify specific HFs. Shown are representative images, N = 7 mice. (b) Additional beginning of catagen, so only 4 Tom+ cells are present during catagen stages. examples of Fig. 1c. Z-stack images of three independent examples of HFs from Yellow dashed line outlines club hair. Dashed white lines outline epithelial- in vivo imaging of Tyr-CreER;Rosa-LSL-tdTomato;K14-rtTA;tetO-H2B-GFP mice dermal boundary. HG: hair germ; Bb: bulb; Bg: bulge; ORSup: upper ORS. ![Figure from page 17](images/page_017_content/img-000.png) Article Extended Data Fig. 3 | Distribution of McSCs in Dct-LacZ mice during catagen the HG. Shown are representative images, N = 5 mice. Dashed black lines outline stages. Images of hair follicles from Dct-LacZ mice at indicated catagen stages. epithelial-dermal boundary. HG: hair germ; Bg: bulge; ORSup: upper ORS. Scale Note that LacZ+ McSCs come together at late catagen to eventually aggregate in bar: 20 μm. ![Figure from page 18](images/page_018_content/img-000.png) Extended Data Fig. 4 | Quantification of dendrite state of McSCs by live (b) Percentages of post-mitotic McSCs with or without both dendritic imaging. Quantification of live imaging experiment in Fig. 2c. (a) Percentages daughters. N = 26 HFs from 2 mice. (c) Percentages of McSCs that migrate of dendritic McSCs prior to or after initial division. N = 31 HFs from 2 mice. before or after dendrite formation. N = 35 HFs from 2 mice. ![Figure from page 19](images/page_019_content/img-000.png) Article Extended Data Fig. 5 | See next page for caption. Extended Data Fig. 5 | Early anagen McSCs activate dendrite and their cell-cycle score. The S and G2M phase cells are grouped together into pigmentation related genes while retaining the expression of SC-enriched proliferative early anagen McSCs and the G1 phase cells constitute the non- genes. (a) FACS plots illustrating the gating strategy to isolate GFP+ melanocytes proliferative early anagen McSCs. (e) Violin plots showing expression of from Dct-rtTA; tetO-H2B-GFP mice. UMAP plot of merged scRNA-seq results from pigmentation genes in the proliferative (Prolif) and non-proliferative sorted telogen McSCs (blue), anagen II McSCs (Early anagen McSC, red) and (Non-Prolif) early anagen McSCs. (f) Featureplots showing expression of anagen VI bulb Mcs (Differentiated bulb Mc, green). (b) Featureplots showing indicated SC-enriched genes in telogen McSCs, early anagen McSCs and late regions and levels of expression of indicated dendrite-related genes. (c) GSEA of anagen bulb Mcs. (g) Quantitative RT-PCR analysis of indicated SC-enriched Gene Ontology Biological Process (GOBP) dendrite development and dendrite genes in bulb Mcs and telogen McSCs. N = 5 mice. Data are represented as morphogenesis in differentially expressed genes (DEGs) from early anagen mean ± SEM. P-values by two-tailed unpaired t test. (h) UMAP plot with dots McSCs compared to telogen McSCs. (d) Top, violin plots of the cell cycle scores of and lines showing pseudo-time trajectory predicted using slingshot package. early anagen McSCs. Bottom, UMAP plot of early anagen McSCs clustered by ![Figure from page 21](images/page_021_content/img-000.png) Article Extended Data Fig. 6 | Examination of the early anagen HG scRNAseq dataset for proliferation and expression of SC-enriched genes. UMAP of early anagen McSCs after cell cycle genes regression. Featureplots of proliferation genes and SC-enriched genes in Oca2− and Oca2+ early anagen McSCs. ![Figure from page 22](images/page_022_content/img-000.png) Extended Data Fig. 7 | See next page for caption. Article Extended Data Fig. 7 | Characterization of Tomato+ cells in Oca2-CreER;Rosa- (blue) in mid/late anagen bulb. (h) Quantification of Tomato+/Dct+ cells. N = 3 LSL-tdTomato mice during lineage tracing. (a) Targeting vector design of mice in anagen onset HG and telogen HG groups and N = 4 mice in mid/late Oca2-CreERT2 knock-in mouse. (b) Cryo section image of Oca2-CreER;Rosa-LSL- anagen bulge/ORSup and bulb groups with at least 10 HFs analyzed per mouse per tdTomato mouse back skin without TAM treatment at 6–13 weeks old. Shown is compartment. Data are represented as mean ± SD. (i) Immunofluorescence of representative image of 4 mice. (c) Experimental design for panels (d)-(j). tdTomato and in situ hybridized Gpr143 probe, Col12a1 probe and Txnip probe. Lineage tracing of Oca2+ HG McSCs in Oca2-CreER; Rosa-LSL-tdTomato HFs. Col12a1 and Txnip were also observed in unrelated epithelial cells. IF shows (d) Z-stack image of Oca2+ (Tom+) cell in whole hair follicle at anagen (Ana) onset. tdTomato and e-cadherin staining with boxed single color enlargements. Analysis of the location of Oca2+ (Tom+) cells in indicated niche compartments at Schematic illustration of differentiation genes (Oca2, Gpr143) and SC-enriched anagen onset. N = 4 mice with at least 10 HFs analyzed per mice. (e) Merged or genes (Col12a, Txnip, E-Cadherin) expression in an early anagen Oca2+ cell and separate immunofluorescence images of Dct (white), Tomato (red), P-cadherin its progeny at subsequent hair cycle stages. Shown are representative images, (green) and DAPI (blue) at early anagen. (f) Immunofluorescence of Tomato and N = 3 mice. (j) Z-stack image of trace labeled Tom+ cells in subsequent telogen. Ki67 and Dct and Ki67 in early anagen. Lower panels depict magnified single Quantification of the location of Tom+ cells in indicated niche compartments in color images in specified regions. Quantification of Ki67+/Tomato+ cells or Ki67+/ subsequent telogen. N = 3 mice with at least 5 HFs analyzed per mice. Dashed Dct+ cells. N = 4 mice in each group with at least 10 HFs analyzed per mouse. Data white lines outline epithelial-dermal boundary. Scale bar: 20 μm, 10 μm in are represented as mean ± SD. P-values by two-tailed unpaired t test. (g) Top: separate channel images. IF: immunofluorescence; FISH: fluorescence in situ Merged or separate immunofluorescence images of Dct (white), Tomato (red), hybridization; HG: hair germ; Bb: bulb; Bg: bulge; ORSup: upper ORS; TAM: Cd34 (green) and DAPI (blue) in mid/late anagen bulge/ORSup. Bottom: Merged tamoxifen; d: day. or separate immunofluorescence images of Dct (green), Tomato (red) and DAPI ![Figure from page 24](images/page_024_content/img-000.png) Extended Data Fig. 8 | Genetic lineage tracing shows that Oca2+ bulb Mcs transcriptional localization of endogenous Oca2 mRNA in late anagen bulge/ do not home back to the HG. (a) Experimental design for lineage tracing of ORSup and bulb from TAM-untreated mice. Shown are representative images Oca2+ late anagen bulb Mcs in Oca2-CreER;Rosa-LSL-tdTomato HFs. (b) Left, of 3 mice. Note that the pattern of endogenous Oca2 mRNA is consistent with immunofluorescence images showing Dct and Tomato (Oca2) expression in Oca2-CreER activity (Tomato signals) in (b) left panels. (d) Schematic illustration McSCs and Mcs in late anagen (depilation 14d) bulge/ORSup and bulb. Right of the fate of Tomato labeled bulb Mcs. Dashed white lines outline epithelial- (after arrow), immunofluorescence image showing lack of Tomato labeling in dermal boundary. IF: immunofluorescence; FISH: fluorescence in situ Dct+ McSCs at subsequent telogen (depilation 25d). Shown are representative hybridization. HG: hair germ; Bg: bulge; Bb: bulb; ORSup: upper ORS; TAM: images, N = 3 mice. (c) Immunofluorescence images of Dct and in situ tamoxifen; d: day. Scale bar: 20 μm. ![Figure from page 25](images/page_025_content/img-000.png) Article Extended Data Fig. 9 | New McSCs become Oca2+ at each hair cycle. single and multiple TAM-injected mice at 3rd telogen. N = 3 mice with at least (a) Experimental scheme of TAM injection during one hair cycle (single TAM) or 3 HFs containing Tom+ cells analyzed per mice. Data are represented as mean ± several continuous hair cycles (repeated TAM) in Oca2-CreER;Rosa-LSL-tdTomato SD. P-value by two-tailed unpaired t test. (f) Z-stack image of Tomato and Dct mice. (b-c) Z-stack images of trace-labeled Tom+ cells in 3rd telogen HFs after co-immunofluorescence at 3rd telogen following repeated TAM injections. single (b) or repeated (c) TAM injections. (d) Number of HFs containing Tom+ To show the co-localization of green and red signals in all McSCs throughout cells in HG after single or multiple TAM injections. HFs were counted at 1st telogen the z-stack, the right top panels show the z-stack of first 15 stacks and the right (green), 2nd telogen (blue) and 3rd telogen (orange). N = 3 mice, with at least 25 HFs bottom panels show the z-stack of last 15 stacks. Shown are representative analyzed per mice per time point. Data are represented as mean ± SD. P-value by images of 3 mice. (g) Schematic summary. Dashed white lines outline epithelial- two-way ANOVA with Bonferroni multiple comparison test. 95% confidence dermal boundary. Scale bar: 20 μm. TAM: tamoxifen; d: day. interval: −94.18 to −6.799. (e) Number of Tomato+ McSC per hair follicle from ![Figure from page 26](images/page_026_content/img-000.png) Extended Data Fig. 10 | See next page for caption. Article Extended Data Fig. 10 | Reversible differentiation of pigmented McSCs mice. (g) Z-stack image of trace-labeled Tom+ cells and corresponding following UVB irradiation. (a) Experimental scheme: Oca2-CreER;Rosa-LSL- brightfield image in telogen HF from UVB-treated mouse at 2 years. Number of tdTomato mice were irradiated with UVB for 3 times and injected with TAM for Tom+ cells per hair follicle after UVB irradiation at mid-anagen and 2 years. 3 times during anagen onset and analyzed at the following time points. (b) Z-stack N = 3 mice with 20 HFs analyzed per mice. (h) Z-stack image of trace-labeled and corresponding brightfield images (separate and merged) of Tom+ cells in Tom+ cells and corresponding brightfield image in anagen VI HF bulb region mid-anagen control and UVB-treated HFs. (c) Number of Tom+ cells in bulge/ from UVB-treated mouse at 2 years. (i) Immunofluorescence of tdTomato ORSup per HF; percentages of pigmented Tom+ cells in bulge/ORSup per HF at and in situ hybridized Col12a1 probe and Txnip probe. IF shows tdTomato mid-anagen. N = 3 mice. 20 HFs analyzed per mice. (d) Immunofluorescence of and e-cadherin staining with boxed single color enlargements. Shown are tdTomato and in situ hybridized Oca2 probe and Gpr143 probe. Quantification representative images, N = 3 mice. ( j) Schematic postulates that all McSCs of the FISH intensity of Oca2 or Gpr143 probe expression in Tomato+ cells in ORSup can undergo differentiation to make pigment after UV treatment, followed by relative to the FISH intensity of Tomato+ cells in the bulb analyzed by HALO FISH reversion to an undifferentiated state, then self-renewal and regeneration of module. N = 3 mice with at least 20 Tom+ cells analyzed per compartment per differentiated Mcs over time. Dashed white lines outline epithelial-dermal mouse. (e) Immunofluorescence images show localization of tdTomato and boundary. Scale bar: 20 μm, 10 μm in separate channel images. Data are differentiation marker Tyrp1 at indicated time points. Percentages of Tyrp1+/ represented as mean ± SD in (c), (d), (e) and (g). P-values by two-tailed unpaired t Tom+ cells in the bulge/ORSup. N = 3 mice. 10 HFs analyzed per mouse. (f) Z-stack test in (c), (d), (e) and (g). IF: immunofluorescence; FISH: fluorescence in situ image of Tom+ cells with corresponding brightfield and Tyrp1 staining for UVB- hybridization; HG: hair germ; Bb: bulb; ORSup: upper ORS; TAM: tamoxifen. treated mice at subsequent telogen. Shown are representative images, N = 3 ![Figure from page 28](images/page_028_content/img-000.png) Extended Data Fig. 11 | See next page for caption. Article Extended Data Fig. 11 | McSCs reversibly enter distinct differentiation nuclear β-catenin+Tomato+ cells. N = 3 mice in early anagen HG and N = 4 mice states correlating with WNT signaling. (a) WNT activation pattern in McSCs in mid/late anagen bulge/ORSup and bulb groups with at least 10 HFs analyzed as previously reported7. (b) VlnPlots show expression of WNT genes in telogen per compartment per mouse. Data are represented as mean ± SD. P-values by McSCs and early anagen McSCs. (c) VlnPlots show expression of WNT genes in ordinary one-way ANOVA with Bonferroni multiple comparison test. 95% Oca2+ early anagen McSCs and Oca2– early anagen McSCs. For (b), see Fig. 2e for confidence intervals: 79.21 to 96.66. (f) Schematic summary of reversible related UMAP. For (c), see Fig. 2g for related UMAP. (d) Experimental scheme of differentiation model. McSCs in the HG during the quiescent telogen phase are Oca2-CreER;Rosa-LSL-tdTomato mice during normal hair cycle. Double-detection in a stem-like state. At anagen onset (regeneration initiation), they undergo of tdTomato and β-catenin in normal hair cycle shown with enlarged single-color activation and differentiation to a TA-like (intermediate) state via WNT images of the denoted areas. Percentages of nuclear β-catenin+Tomato+ cells. activation. These cells then either fully differentiate into terminal Mcs or revert N = 3 mice with at least 10 HFs analyzed per compartment per mouse. Data to a stem-like state upon migration into the WNT-negative bulge/ORSup. are represented as mean ± SD. P-values by ordinary one-way ANOVA with Subsequently, most McSCs home back to the HG. Orange dotted lines in single Bonferroni multiple comparison test. 95% confidence intervals: 83.22 to 98.56. color images suggest nuclei location. Dashed white lines outline epithelial- (e) Experimental scheme of Oca2-CreER;Rosa-LSL-tdTomato mice with UVB dermal boundary. Scale bar: 20 μm, 10 μm in separate channel images. IF: irradiation. Double-detection of tdTomato and β-catenin with UVB treatment immunofluorescence; HG: hair germ; Bg: bulge; Bb: bulb; ORSup: upper ORS; shown with enlarged single-color images of the denoted areas. Percentages of TAM: tamoxifen; d: day; SC: stem cell; TA: transit-amplifying; DF: differentiated. ![Figure from page 30](images/page_030_content/img-000.png) Extended Data Fig. 12 | Constitutive WNT activation induces pigmentation mouse/area. (c) Evaluation of how long-term constitutive WNT activation of Oca2+ McSCs, leading to their loss. (a) Experimental scheme: Oca2-CreER; affects the maintenance of Oca2+ McSCs. Z-stack images of Tom+ cells at Rosa-LSL-tdTomato control and Oca2-CreER;β-catenin-STA;Rosa-LSL-tdTomato 7 months post-TAM injection. Number of Tom+ McSCs per hair follicle in the mice were depilated and injected with TAM for 7 days. (b) Evaluation of telogen bulge/HG. Data are represented as mean ± SD. P-value by two-tailed pigmentation status of Tom+ McSCs with or without constitutive WNT unpaired t test. N = 4 mice in control, N = 4 independent areas from 2 mice in activation. Z-stack images of Tom+ cells and corresponding brightfield images β-catenin-STA group. At least 15 HFs analyzed per mouse/area. Dashed lines at 1 day after last TAM injection. Percentages of pigmented Tom+ McSCs. Data outline epithelial-dermal boundary. Arrowheads point to Tom+ McSCs in the are represented as mean ± SD. P-values by two-tailed unpaired t test. N = 4 mice bulge/ORSup. Scale bar: 20 μm. HG: hair germ; Bg: bulge; ORSup: upper ORS; (control), N = 2 mice, 3 independent areas (β-catenin-STA). 30 HFs analyzed per TAM: tamoxifen; d: day. ![Figure from page 31](images/page_031_content/img-000.png) Article Extended Data Fig. 13 | Increased scattering of McSCs to the bulge in aged telogen. (e) Quantification of Tom+ McSCs residing in the HG or bulge in Dct-LacZ and Oca2-CreER;Rosa-LSL-tdTomato mice. (a) Experimental Oca2-CreER;Rosa-LSL-tdTomato mice at indicated time points. N = 42 HFs from scheme of repeated plucking of Dct-LacZ mice. (b) Images of hair follicles from 2 mice at 3 months. N = 54 HFs from 2 mice at 14 months. Dashed lines outline Dct-LacZ mice at 2nd telogen and 7th telogen after hair plucking. (c) Experimental epithelial-dermal boundary. Solid circles suggest location of dermal papilla. scheme: early repeated plucking followed by natural aging of Oca2-CreER; Scale bar: 20 μm. HG: hair germ; Bg: bulge; TAM: tamoxifen; d: day; PN: Rosa-LSL-tdTomato mice. (d) Z-stack images of hair follicles from Oca2-CreER; postnatal; Telo: telogen. Rosa-LSL-tdTomato mice at 3 months old (3rd telogen) and 14 months old ![Figure from page 32](images/page_032_content/img-000.png) ʹ
    β
        β
β

## Document Summary

This document was converted from **07_Psychology_Emotion/Books/Dedifferentiation maintains melanocyte stem cells in a dynamic niche - s41586-023-05960-6.pdf** containing 35 pages.

**Usage with Claude Code:**

bash claude code "Collections/07_Psychology_Emotion_Books_Dedifferentiation_maintains_melanocyte_stem_cells_in_a_dynamic_niche_-_s41586-023-05960-6/07_Psychology_Emotion_Books_Dedifferentiation_maintains_melanocyte_stem_cells_in_a_dynamic_niche_-_s41586-023-05960-6.md" ```

**Library Location:** 07_Psychology_Emotion is organized as:
- **Books/**: Complete texts and manuals
- **Articles/**: Papers, documents, and shorter works

*Conversion completed: Mon Aug 18 07:08:14 PM PDT 2025*
